<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Computation of melting temperature of nucleic acid duplexes with `rmelting` • rmelting</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.4.0/flatly/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><!-- docsearch --><script src="../docsearch.js"></script><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/docsearch.js/2.6.3/docsearch.min.css" integrity="sha256-QOSRU/ra9ActyXkIBbiIB144aDBdtvXBcNc3OTNuX/Q=" crossorigin="anonymous">
<link href="../docsearch.css" rel="stylesheet">
<script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js" integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin="anonymous"></script><meta property="og:title" content="Computation of melting temperature of nucleic acid duplexes with `rmelting`">
<meta property="og:description" content="rmelting">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--><!-- Global site tag (gtag.js) - Google Analytics --><script async src="https://www.googletagmanager.com/gtag/js?id=UA-116683292-2"></script><script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-116683292-2');
</script>
</head>
<body data-spy="scroll" data-target="#toc">
    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">rmelting</a>
        <span class="version label label-danger" data-toggle="tooltip" data-placement="bottom" title="Unreleased version">1.7.1</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../index.html">
    <span class="fas fa-home fa-lg"></span>
     
    Home
  </a>
</li>
<li>
  <a href="../reference/index.html">
    <span class="fas fa-code"></span>
     
    Reference
  </a>
</li>
<li>
  <a href="../articles/Tutorial.html">
    <span class="fas fa-file-o"></span>
     
    Introduction
  </a>
</li>
<li>
  <a href="../news/index.html">
    <span class="fas fa-newspaper-o"></span>
     
    News
  </a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/aravind-j/rmelting">
    <span class="fas fa-github-alt"></span>
     
    Github
  </a>
</li>
      </ul>
<form class="navbar-form navbar-right hidden-xs hidden-sm" role="search">
        <div class="form-group">
          <input type="search" class="form-control" name="search-input" id="search-input" placeholder="Search..." aria-label="Search for..." autocomplete="off">
</div>
      </form>
      
    </div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><script src="Tutorial_files/accessible-code-block-0.0.1/empty-anchor.js"></script><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>Computation of melting temperature of nucleic acid duplexes with <code>rmelting</code>
</h1>
                        <h4 class="author">Aravind, J.<sup>1</sup> and Krishna, G. K.<sup>2</sup>
</h4>
            
            <h4 class="date">2021-03-04</h4>
      
      <small class="dont-index">Source: <a href="https://github.com/aravind-j/rmelting/blob/master/vignettes/Tutorial.Rmd"><code>vignettes/Tutorial.Rmd</code></a></small>
      <div class="hidden name"><code>Tutorial.Rmd</code></div>

    </div>

    
    
<p>
</p>
<ol style="list-style-type: decimal">
<li>Division of Germplasm Conservation, ICAR-National Bureau of Plant Genetic Resources, New Delhi.
</li>
</ol>
<p>
</p>
<ol start="2" style="list-style-type: decimal">
<li>Division of Genetics, ICAR-Indian Agricultural Research Institute, New Delhi.
</li>
</ol>
<p><img src="https://raw.githubusercontent.com/aravind-j/rmelting/master/inst/extdata/rmelting.png" align="right" alt="logo" width="173" height="200" style="padding: 10px; border: none; float: right;"></p>
<div id="introduction" class="section level1">
<h1 class="hasAnchor">
<a href="#introduction" class="anchor"></a>Introduction</h1>
<p>The <code>R</code> package <code>rmelting</code> is an interface to the <a href="https://www.ebi.ac.uk/biomodels/tools/melting/">MELTING 5</a> program <span class="citation">(Le Novère, 2001; Dumousseau et al., 2012)</span> to compute melting temperatures of nucleic acid duplexes (DNA/DNA, DNA/RNA, RNA/RNA or 2’-O-MeRNA/RNA) along with other thermodynamic parameters such as hybridisation enthalpy and entropy.</p>
<p>Melting temperatures are computed by Nearest-neighbour methods for short sequences or approximative estimation formulae for long sequences. Apart from these, multiple corrections are available to take into account the presence of Cations (Na, Tris, K and Mg) or denaturing agents (DMSO and formamide).</p>
</div>
<div id="installation" class="section level1">
<h1 class="hasAnchor">
<a href="#installation" class="anchor"></a>Installation</h1>
<p>The package can be installed from Bioconductor as follows.</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="kw">if</span> <span class="op">(</span><span class="op">!</span><span class="st">"BiocManager"</span> <span class="op">%in%</span> <span class="fu"><a href="https://rdrr.io/r/base/colnames.html">rownames</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/utils/installed.packages.html">installed.packages</a></span><span class="op">(</span><span class="op">)</span><span class="op">)</span><span class="op">)</span> 
  <span class="fu"><a href="https://rdrr.io/r/utils/install.packages.html">install.packages</a></span><span class="op">(</span><span class="st">"BiocManager"</span><span class="op">)</span>
<span class="fu">BiocManager</span><span class="fu">::</span><span class="fu">install</span><span class="op">(</span><span class="st">"rmelting"</span><span class="op">)</span></code></pre></div>
<p>The development version can be installed from github as follows.</p>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="kw">if</span> <span class="op">(</span><span class="op">!</span><span class="kw"><a href="https://rdrr.io/r/base/library.html">require</a></span><span class="op">(</span><span class="st">'devtools'</span><span class="op">)</span><span class="op">)</span> <span class="fu"><a href="https://rdrr.io/r/utils/install.packages.html">install.packages</a></span><span class="op">(</span><span class="st">'devtools'</span><span class="op">)</span>
<span class="fu">devtools</span><span class="fu">::</span><span class="fu">install_github</span><span class="op">(</span><span class="st">"aravind-j/rmelting"</span><span class="op">)</span></code></pre></div>
<p>Then the package can be loaded as follows.</p>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/aravind-j/rmelting">rmelting</a></span><span class="op">)</span></code></pre></div>
</div>
<div id="basic-usage" class="section level1">
<h1 class="hasAnchor">
<a href="#basic-usage" class="anchor"></a>Basic usage</h1>
<p>Melting temperatures are computed in <code>rmelting</code> through the core function <code>melting</code> which takes a number of arguments (see <code><a href="../reference/melting.html">?melting</a></code>). The following are the essential arguments which are mandatory for computation.</p>
<ul>
<li>
<code>sequence</code>
<ul>
<li>5’ to 3’ sequence of one strand of the nucleic acid duplex as a character string. Recognises A, C, G, T, U, I, X_C, X_T, A*, AL, TL, GL and CL (<strong>Table 1</strong>). U and T are not considered identical.</li>
</ul>
</li>
</ul>
<p><strong>Table 1</strong>: Recognized sequences</p>
<table class="table">
<thead><tr class="header">
<th align="left">Code</th>
<th align="left">Type</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">A</td>
<td align="left">Adenine</td>
</tr>
<tr class="even">
<td align="left">C</td>
<td align="left">Cytosine</td>
</tr>
<tr class="odd">
<td align="left">G</td>
<td align="left">Guanine</td>
</tr>
<tr class="even">
<td align="left">T</td>
<td align="left">Thymine</td>
</tr>
<tr class="odd">
<td align="left">U</td>
<td align="left">Uracil</td>
</tr>
<tr class="even">
<td align="left">I</td>
<td align="left">Inosine</td>
</tr>
<tr class="odd">
<td align="left">X_C</td>
<td align="left">Trans azobenzenes</td>
</tr>
<tr class="even">
<td align="left">X_T</td>
<td align="left">Cis azobenzenes</td>
</tr>
<tr class="odd">
<td align="left">A*</td>
<td align="left">Hydroxyadenine</td>
</tr>
<tr class="even">
<td align="left">AL</td>
<td align="left">Locked nucleic acid</td>
</tr>
<tr class="odd">
<td align="left">TL</td>
<td align="left">’’</td>
</tr>
<tr class="even">
<td align="left">GL</td>
<td align="left">’’</td>
</tr>
<tr class="odd">
<td align="left">CL</td>
<td align="left">’’</td>
</tr>
</tbody>
</table>
<ul>
<li>
<code>Comp.sequence</code>
<ul>
<li>Mandatory if there are mismatches, inosine(s) or hydroxyadenine(s) between the two strands. If not specified, it is computed as the complement of <code>sequence</code>. Self-complementarity in <code>sequence</code> is detected even though there may be (are) dangling end(s) and <code>comp.sequence</code> is computed.</li>
</ul>
</li>
<li>
<code>nucleic.acid.conc</code>
<ul>
<li>In molar concentration (M or mol L<sup>-1</sup>).</li>
</ul>
</li>
<li>
<code>Na.conc, Mg.conc, Tris.conc, K.conc</code>
<ul>
<li>At least one cation (Na, Mg, Tris, K) concentration is mandatory, the other agents(dNTP, DMSO, formamide) are optional.</li>
</ul>
</li>
<li>
<code>hybridisation.type</code>
<ul>
<li>The possible options for hybridisation type are as follows (<strong>Table 2</strong>).</li>
</ul>
</li>
</ul>
<p><strong>Table 2</strong>: Hybridisation type options</p>
<table class="table">
<thead><tr class="header">
<th align="left">Option</th>
<th align="left">Sequence</th>
<th align="left">Complementary sequence</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left"><code>dnadna</code></td>
<td align="left">DNA</td>
<td align="left">DNA</td>
</tr>
<tr class="even">
<td align="left"><code>rnarna</code></td>
<td align="left">RNA</td>
<td align="left">RNA</td>
</tr>
<tr class="odd">
<td align="left"><code>dnarna</code></td>
<td align="left">DNA</td>
<td align="left">RNA</td>
</tr>
<tr class="even">
<td align="left"><code>rnadna</code></td>
<td align="left">RNA</td>
<td align="left">DNA</td>
</tr>
<tr class="odd">
<td align="left"><code>mrnarna</code></td>
<td align="left">2-o-methyl RNA</td>
<td align="left">RNA</td>
</tr>
<tr class="even">
<td align="left"><code>rnamrna</code></td>
<td align="left">RNA</td>
<td align="left">2-o-methyl RNA</td>
</tr>
</tbody>
</table>
<p>With these arguments, the melting temperature can be computed as follows.</p>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAGTGAGACAGCAATGGTCG"</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">2e-06</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 73.35168</code></pre>
<p>Only the melting temperature is given as a console output. However, the output can be assigned to an object which contains the details of the environment, options and the thermodynamics results as a list.</p>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Get output as list</span>
<span class="va">out</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAGTGAGACAGCAATGGTCG"</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">2e-06</span>,
               hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span><span class="op">)</span>
<span class="co"># Environment output</span>
<span class="va">out</span><span class="op">$</span><span class="va">Environment</span></code></pre></div>
<pre><code>## $Sequence
## [1] "CAGTGAGACAGCAATGGTCG"
## 
## $`Complementary sequence`
## [1] "GTCACTCTGTCGTTACCAGC"
## 
## $`Nucleic acid concentration (M)`
## [1] 2e-06
## 
## $`Hybridization type`
## [1] "dnadna"
## 
## $`Na concentration (M)`
## [1] 1
## 
## $`Mg concentration (M)`
## [1] 0
## 
## $`Tris concentration (M)`
## [1] 0
## 
## $`K concentration (M)`
## [1] 0
## 
## $`dNTP concentration (M)`
## [1] 0
## 
## $`DMSO concentration (%)`
## [1] 0
## 
## $`Formamide concentration (M or %)`
## [1] 0
## 
## $`Self complementarity`
## [1] FALSE
## 
## $`Correction factor`
## [1] 4</code></pre>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Options used</span>
<span class="va">out</span><span class="op">$</span><span class="va">Options</span></code></pre></div>
<pre><code>## $`Approximative formula`
## [1] NA
## 
## $`Nearest neighbour model`
## [1] "all97"
## 
## $`GU model`
## [1] NA
## 
## $`Single mismatch model`
## [1] "allsanpey"
## 
## $`Tandem mismatch model`
## [1] "allsanpey"
## 
## $`Single dangling end model`
## [1] "bom00"
## 
## $`Double dangling end model`
## [1] "sugdna02"
## 
## $`Long dangling end model`
## [1] "sugdna02"
## 
## $`Internal loop model`
## [1] "san04"
## 
## $`Single bulge loop model`
## [1] "tan04"
## 
## $`Long bulge loop model`
## [1] "san04"
## 
## $`CNG repeats model`
## [1] NA
## 
## $`Inosine bases model`
## [1] "san05"
## 
## $`Hydroxyadenine bases model`
## [1] "sug01"
## 
## $`Azobenzenes model`
## [1] "asa05"
## 
## $`Locked nucleic acids model`
## [1] "mct04"
## 
## $`Ion correction method`
## [1] NA
## 
## $`Na equivalence correction method`
## [1] "ahs01"
## 
## $`DMSO correction method`
## [1] "ahs01"
## 
## $`Formamide correction method`
## [1] "bla96"
## 
## $Mode
## [1] "def"</code></pre>
<div class="sourceCode" id="cb10"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Thermodynamics results</span>
<span class="va">out</span><span class="op">$</span><span class="va">Results</span></code></pre></div>
<pre><code>## $`Enthalpy (cal)`
## [1] -159000
## 
## $`Entropy (cal)`
## [1] -430
## 
## $`Enthalpy (J)`
## [1] -664620
## 
## $`Entropy (J)`
## [1] -1797.4
## 
## $`Melting temperature (C)`
## [1] 73.35168</code></pre>
<p>The command for the MELTING 5 java version is saved as an attribute in the list <code>out</code> and can be retrieved as follows.</p>
<div class="sourceCode" id="cb12"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Command for MELTING 5</span>
<span class="fu"><a href="https://rdrr.io/r/base/attributes.html">attributes</a></span><span class="op">(</span><span class="va">out</span><span class="op">)</span><span class="op">$</span><span class="va">command</span></code></pre></div>
<pre><code>## [1] "-S CAGTGAGACAGCAATGGTCG -H dnadna -P 2e-06 -E Na=1 -T 60"</code></pre>
</div>
<div id="melting-temperature-computation" class="section level1">
<h1 class="hasAnchor">
<a href="#melting-temperature-computation" class="anchor"></a>Melting temperature computation</h1>
<p>Melting temperature is computed by either approximative or nearest neighbour methods according to the length of the oligonucleotide sequences. For longer sequences (longer than the threshold value, the threshold value set by <code>size.threshold</code> with the default value 60) approximative method is used, while for others, nearest neighbour method is used.</p>
<div id="approximative-methods" class="section level2">
<h2 class="hasAnchor">
<a href="#approximative-methods" class="anchor"></a>Approximative methods</h2>
<p>The approximative method for computation can be specified by the argument <code>method.approx</code>. The available methods are given in <strong>Table 3</strong>.</p>
<p><strong>Table 3</strong>: Details of approximative methods</p>
<table class="table">
<colgroup>
<col width="18%">
<col width="11%">
<col width="23%">
<col width="46%">
</colgroup>
<thead><tr class="header">
<th align="left">Formula</th>
<th align="left">Type</th>
<th align="left">Limits/Remarks</th>
<th align="left">Reference</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left"><code>ahs01</code></td>
<td align="left">DNA</td>
<td align="left">No mismatch</td>
<td align="left"><span class="citation">Ahsen et al. (2001)</span></td>
</tr>
<tr class="even">
<td align="left"><code>che93</code></td>
<td align="left">DNA</td>
<td align="left">No mismatch; Na=0, Mg=0.0015, Tris=0.01, K=0.05</td>
<td align="left"><span class="citation">Marmur and Doty (1962)</span></td>
</tr>
<tr class="odd">
<td align="left"><code>che93corr</code></td>
<td align="left">DNA</td>
<td align="left">No mismatch; Na=0, Mg=0.0015, Tris=0.01, K=0.05</td>
<td align="left"><span class="citation">Marmur and Doty (1962)</span></td>
</tr>
<tr class="even">
<td align="left"><code>schdot</code></td>
<td align="left">DNA</td>
<td align="left">No mismatch</td>
<td align="left">
<span class="citation">Wetmur (1991)</span>, <span class="citation">Marmur and Doty (1962)</span>, <span class="citation">Chester and Marshak (1993)</span>, <span class="citation">Schildkraut and Lifson (1965)</span>, <span class="citation">Wahl et al. (1987)</span>, <span class="citation">Britten et al. (1974)</span>, <span class="citation">Hall et al. (1980)</span>
</td>
</tr>
<tr class="odd">
<td align="left"><code>owe69</code></td>
<td align="left">DNA</td>
<td align="left">No mismatch</td>
<td align="left">
<span class="citation">Owen et al. (1969)</span>, <span class="citation">Frank-Kamenetskii (1971)</span>, <span class="citation">Blake (1996)</span>, <span class="citation">Blake and Delcourt (1998)</span>
</td>
</tr>
<tr class="even">
<td align="left"><code>san98</code></td>
<td align="left">DNA</td>
<td align="left">No mismatch</td>
<td align="left">
<span class="citation">SantaLucia (1998)</span>, <span class="citation">Ahsen et al. (2001)</span>
</td>
</tr>
<tr class="odd">
<td align="left">
<code>wetdna91</code>*</td>
<td align="left">DNA</td>
<td align="left"></td>
<td align="left"><span class="citation">Wetmur (1991)</span></td>
</tr>
<tr class="even">
<td align="left">
<code>wetrna91</code>*</td>
<td align="left">RNA</td>
<td align="left"></td>
<td align="left"><span class="citation">Wetmur (1991)</span></td>
</tr>
<tr class="odd">
<td align="left">
<code>wetdnarna91</code>*</td>
<td align="left">DNA/RNA</td>
<td align="left"></td>
<td align="left"><span class="citation">Wetmur (1991)</span></td>
</tr>
</tbody>
</table>
* Default method for computation.

<p><em><strong>Examples</strong></em></p>
<pre><code>    DNA:TCTAATGTGCTGTTAGATGTATCCAGAGATAGCCGAGCATAAACTTCAACACACGAGACGTTGATTGGATTTAACCATAG
        ||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
    DNA:AGATTACACGACAATCTACATAGGTCTCTATCGGCTCGTATTTGAAGTTGTGTGCTCTGCAACTAACCTAAATTGGTATC

    RNA:UUAAUCUCCGUCAUCUUUAAGCCGUGGAGAGACUGUAGACUUGAACAGGGGUAAGCGGAGGCACGUAGGAUUCACAUCAU
        ||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
    RNA:AAUUAGAGGCAGUAGAAAUUCGGCACCUCUCUGACAUCUGAACUUGUCCCCAUUCGCCUCCGUGCAUCCUAAGUGUAGUA
    
    DNA:TCTAATGTGCTGTTAGATGTATCCAGAGATAGCCGAGCATAAACTTCAACACACGAGACGTTGATTGGATTTAACCATAG
        ||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
    RNA:AGAUUACACGACAAUCUACAUAGGUCUCUAUCGGCUCGUAUUUGAAGUUGUGUGCUCUGCAACUAACCUAAAUUGGUAUC
    </code></pre>
<div class="sourceCode" id="cb15"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Long Nucleotide sequence</span>
<span class="va">DNAseq</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span><span class="st">"TCTAATGTGCTGTTAGATGTATCCAGAGATAGCCGAGCATAAACTTCAACACACGAGACGTTGATTGGATTTAACCATAG"</span><span class="op">)</span>
<span class="va">RNAseq</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span><span class="st">"UUAAUCUCCGUCAUCUUUAAGCCGUGGAGAGACUGUAGACUUGAACAGGGGUAAGCGGAGGCACGUAGGAUUCACAUCAU"</span><span class="op">)</span>

<span class="co"># Approximative method - default (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="va">DNAseq</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">2e-06</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 87.82455</code></pre>
<div class="sourceCode" id="cb17"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Approximative method - wetdna91 (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="va">DNAseq</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">2e-06</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span>,
        method.approx <span class="op">=</span> <span class="st">"wetdna91"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 87.82455</code></pre>
<div class="sourceCode" id="cb19"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Approximative method - ahs01 (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="va">DNAseq</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">2e-06</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span>,
        method.approx <span class="op">=</span> <span class="st">"ahs01"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 87.325</code></pre>
<div class="sourceCode" id="cb21"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Approximative method - che93 (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="va">DNAseq</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">2e-06</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span>,
        method.approx <span class="op">=</span> <span class="st">"che93"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 77.575</code></pre>
<div class="sourceCode" id="cb23"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Approximative method - che93corr (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="va">DNAseq</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">2e-06</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span>,
        method.approx <span class="op">=</span> <span class="st">"che93corr"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 79.0125</code></pre>
<div class="sourceCode" id="cb25"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Approximative method - schdot (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="va">DNAseq</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">2e-06</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span>,
        method.approx <span class="op">=</span> <span class="st">"schdot"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 89.4625</code></pre>
<div class="sourceCode" id="cb27"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Approximative method - owe69 (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="va">DNAseq</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">2e-06</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span>,
        method.approx <span class="op">=</span> <span class="st">"owe69"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 100.96</code></pre>
<div class="sourceCode" id="cb29"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Approximative method - san98 (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="va">DNAseq</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">2e-06</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span>,
        method.approx <span class="op">=</span> <span class="st">"san98"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 86.9</code></pre>
<div class="sourceCode" id="cb31"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Approximative method - default (RNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="va">RNAseq</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">2e-06</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 101.1745</code></pre>
<div class="sourceCode" id="cb33"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Approximative method - wetrna91 (RNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="va">RNAseq</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">2e-06</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span>,
        method.approx <span class="op">=</span> <span class="st">"wetrna91"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 101.1745</code></pre>
<div class="sourceCode" id="cb35"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Approximative method - wetdnarna91 (DNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="va">DNAseq</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">2e-06</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"dnarna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 88.92455</code></pre>
<div class="sourceCode" id="cb37"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Approximative method - wetdnarna91 (DNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="va">DNAseq</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">2e-06</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"dnarna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span>,
        method.approx <span class="op">=</span> <span class="st">"wetdnarna91"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 88.92455</code></pre>
</div>
<div id="nearest-neighbour-methods" class="section level2">
<h2 class="hasAnchor">
<a href="#nearest-neighbour-methods" class="anchor"></a>Nearest neighbour methods</h2>
<div id="perfectly-matching-sequences" class="section level3">
<h3 class="hasAnchor">
<a href="#perfectly-matching-sequences" class="anchor"></a>Perfectly matching sequences</h3>
<p>The nearest neighbour model for computation in case of perfectly matching sequences can be specified by the argument <code>method.nn</code>. The available methods are given in <strong>Table 4</strong>.</p>
<p><strong>Table 4</strong>: Details of nearest neighbour methods for perfectly matching sequences</p>
<table class="table">
<colgroup>
<col width="13%">
<col width="17%">
<col width="27%">
<col width="41%">
</colgroup>
<thead><tr class="header">
<th align="left">Model</th>
<th align="left">Type</th>
<th align="left">Limits/Remarks</th>
<th align="left">Reference</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">
<code>all97</code>*</td>
<td align="left">DNA</td>
<td align="left"></td>
<td align="left"><span class="citation">Allawi and SantaLucia (1997)</span></td>
</tr>
<tr class="even">
<td align="left"><code>bre86</code></td>
<td align="left">DNA</td>
<td align="left"></td>
<td align="left"><span class="citation">Breslauer et al. (1986)</span></td>
</tr>
<tr class="odd">
<td align="left"><code>san04</code></td>
<td align="left">DNA</td>
<td align="left"></td>
<td align="left"><span class="citation">SantaLucia and Hicks (2004)</span></td>
</tr>
<tr class="even">
<td align="left"><code>san96</code></td>
<td align="left">DNA</td>
<td align="left"></td>
<td align="left"><span class="citation">SantaLucia et al. (1996)</span></td>
</tr>
<tr class="odd">
<td align="left"><code>sug96</code></td>
<td align="left">DNA</td>
<td align="left"></td>
<td align="left"><span class="citation">Sugimoto et al. (1996)</span></td>
</tr>
<tr class="even">
<td align="left"><code>tan04</code></td>
<td align="left">DNA</td>
<td align="left"></td>
<td align="left"><span class="citation">Tanaka et al. (2004)</span></td>
</tr>
<tr class="odd">
<td align="left"><code>fre86</code></td>
<td align="left">RNA</td>
<td align="left"></td>
<td align="left"><span class="citation">Freier et al. (1986)</span></td>
</tr>
<tr class="even">
<td align="left">
<code>xia98</code>*</td>
<td align="left">RNA</td>
<td align="left"></td>
<td align="left"><span class="citation">Xia et al. (1998)</span></td>
</tr>
<tr class="odd">
<td align="left">
<code>sug95</code>*</td>
<td align="left">DNA/ RNA</td>
<td align="left"></td>
<td align="left"><span class="citation">SantaLucia et al. (1996)</span></td>
</tr>
<tr class="even">
<td align="left">
<code>tur06</code>*</td>
<td align="left">2’-O-MeRNA/ RNA</td>
<td align="left">A sodium correction (<code>san04</code>) is automatically applied to convert the entropy (Na = 0.1M) into the entropy (Na = 1M)</td>
<td align="left"><span class="citation">Kierzek et al. (2006)</span></td>
</tr>
</tbody>
</table>
* Default method for computation.

<p><em><strong>Examples</strong></em></p>
<pre><code>    DNA:CAGTGAGACAGCAATGGTCG
        ||||||||||||||||||||
    DNA:GTCACTCTGTCGTTACCAGC

    RNA:CAGUGAGACAGCAAUGGUCG
        ||||||||||||||||||||
    RNA:GUCACUCUGUCGUUACCAGC
    
    DNA:CAGTGAGACAGCAATGGTCG
        ||||||||||||||||||||
    RNA:GUCACUCUGUCGUUACCAGC
    </code></pre>
<div class="sourceCode" id="cb40"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Nearest neighbour method - default (DNA/DNA: No Self-Complimentarity)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAGTGAGACAGCAATGGTCG"</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">2e-06</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 73.35168</code></pre>
<div class="sourceCode" id="cb42"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Nearest neighbour method - all97 (DNA/DNA: No Self-Complimentarity)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAGTGAGACAGCAATGGTCG"</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">2e-06</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span>, method.nn <span class="op">=</span> <span class="st">"all97"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 73.35168</code></pre>
<div class="sourceCode" id="cb44"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Nearest neighbour method - bre86 (DNA/DNA: No Self-Complimentarity)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAGTGAGACAGCAATGGTCG"</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">2e-06</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span>, method.nn <span class="op">=</span> <span class="st">"bre86"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 83.2203</code></pre>
<div class="sourceCode" id="cb46"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Nearest neighbour method - san04 (DNA/DNA: No Self-Complimentarity)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAGTGAGACAGCAATGGTCG"</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">2e-06</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span>, method.nn <span class="op">=</span> <span class="st">"san04"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 73.30191</code></pre>
<div class="sourceCode" id="cb48"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Nearest neighbour method - san96 (DNA/DNA: No Self-Complimentarity)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAGTGAGACAGCAATGGTCG"</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">2e-06</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span>, method.nn <span class="op">=</span> <span class="st">"san96"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 75.7102</code></pre>
<div class="sourceCode" id="cb50"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Nearest neighbour method - sug96 (DNA/DNA: No Self-Complimentarity)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAGTGAGACAGCAATGGTCG"</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">2e-06</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span>, method.nn <span class="op">=</span> <span class="st">"sug96"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 78.17556</code></pre>
<div class="sourceCode" id="cb52"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Nearest neighbour method - tan04 (DNA/DNA: No Self-Complimentarity)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAGTGAGACAGCAATGGTCG"</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">2e-06</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span>, method.nn <span class="op">=</span> <span class="st">"tan04"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 71.31413</code></pre>
<div class="sourceCode" id="cb54"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Nearest neighbour method - default (RNA/RNA: No Self-Complimentarity)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAGUGAGACAGCAAUGGUCG"</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">2e-06</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 86.77685</code></pre>
<div class="sourceCode" id="cb56"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Nearest neighbour method - xia98 (RNA/RNA: No Self-Complimentarity)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAGUGAGACAGCAAUGGUCG"</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">2e-06</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span>, method.nn <span class="op">=</span> <span class="st">"xia98"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 86.77685</code></pre>
<div class="sourceCode" id="cb58"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Nearest neighbour method - fre86 (RNA/RNA: No Self-Complimentarity)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAGUGAGACAGCAAUGGUCG"</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">2e-06</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span>, method.nn <span class="op">=</span> <span class="st">"fre86"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 83.81257</code></pre>
<div class="sourceCode" id="cb60"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Nearest neighbour method - default (mRNA/RNA: No Self-Complimentarity)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAGUGAGACAGCAAUGGUCG"</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">2e-06</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"mrnarna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 99.01986</code></pre>
<div class="sourceCode" id="cb62"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Nearest neighbour method - tur06 (mRNA/RNA: No Self-Complimentarity)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAGUGAGACAGCAAUGGUCG"</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">2e-06</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"mrnarna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span>, method.nn <span class="op">=</span> <span class="st">"tur06"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 99.01986</code></pre>
<div class="sourceCode" id="cb64"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Nearest neighbour method - default (DNA/RNA: No Self-Complimentarity)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAGTGAGACAGCAATGGTCG"</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">2e-06</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"dnarna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 66.77049</code></pre>
<div class="sourceCode" id="cb66"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Nearest neighbour method - sug95 (DNA/RNA: No Self-Complimentarity)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAGTGAGACAGCAATGGTCG"</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">2e-06</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"dnarna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span>, method.nn <span class="op">=</span> <span class="st">"sug95"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 66.77049</code></pre>
<p>Self complementarity for perfect matching sequences or sequences with dangling ends is detected automatically. However it can be enforced by the argument <code>force.self = TRUE</code>.</p>
<p><em><strong>Examples</strong></em></p>
<pre><code>    DNA:CATATGGCCATATG
        ||||||||||||||
    DNA:GTATACCGGTATAC

    RNA:AUGUACAU
        ||||||||
    RNA:UACAUGUA</code></pre>
<div class="sourceCode" id="cb69"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Nearest neighbour method - default (DNA/DNA: Self-Complimentarity)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CATATGGCCATATG"</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">2e-06</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 56.00644</code></pre>
<div class="sourceCode" id="cb71"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Nearest neighbour method - all97 (DNA/DNA: Self-Complimentarity)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CATATGGCCATATG"</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">2e-06</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span>, method.nn <span class="op">=</span> <span class="st">"all97"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 56.00644</code></pre>
<div class="sourceCode" id="cb73"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Nearest neighbour method - bre86 (DNA/DNA: Self-Complimentarity)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CATATGGCCATATG"</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">2e-06</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span>, method.nn <span class="op">=</span> <span class="st">"bre86"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 63.44605</code></pre>
<div class="sourceCode" id="cb75"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Nearest neighbour method - san04 (DNA/DNA: Self-Complimentarity)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CATATGGCCATATG"</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">2e-06</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span>, method.nn <span class="op">=</span> <span class="st">"san04"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 57.80792</code></pre>
<div class="sourceCode" id="cb77"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Nearest neighbour method - san96 (DNA/DNA: Self-Complimentarity)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CATATGGCCATATG"</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">2e-06</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span>, method.nn <span class="op">=</span> <span class="st">"san96"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 55.0921</code></pre>
<div class="sourceCode" id="cb79"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Nearest neighbour method - sug96 (DNA/DNA: Self-Complimentarity)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CATATGGCCATATG"</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">2e-06</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span>, method.nn <span class="op">=</span> <span class="st">"sug96"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 59.06213</code></pre>
<div class="sourceCode" id="cb81"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Nearest neighbour method - tan04 (DNA/DNA: Self-Complimentarity)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CATATGGCCATATG"</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">2e-06</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span>, method.nn <span class="op">=</span> <span class="st">"tan04"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 55.65824</code></pre>
<div class="sourceCode" id="cb83"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Nearest neighbour method - default (RNA/RNA: Self-Complimentarity)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"AUGUACAU"</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">2e-06</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 30.27015</code></pre>
<div class="sourceCode" id="cb85"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Nearest neighbour method - xia98 (RNA/RNA: Self-Complimentarity)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"AUGUACAU"</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">2e-06</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span>, method.nn <span class="op">=</span> <span class="st">"xia98"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 30.27015</code></pre>
<div class="sourceCode" id="cb87"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Nearest neighbour method - fre86 (RNA/RNA: Self-Complimentarity)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"AUGUACAU"</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">2e-06</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span>, method.nn <span class="op">=</span> <span class="st">"fre86"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 31.48175</code></pre>
</div>
<div id="gu-wobble-base-pairs-effect" class="section level3">
<h3 class="hasAnchor">
<a href="#gu-wobble-base-pairs-effect" class="anchor"></a>GU wobble base pairs effect</h3>
<p>The nearest neighbour model for computation in case of sequences with GU wobble base pairs can be specified by the argument <code>method.GU</code>. The available methods are given in <strong>Table 5</strong>.</p>
<p><strong>Table 5</strong>: Details of methods for sequences with GU wobble base pairs</p>
<table class="table">
<colgroup>
<col width="15%">
<col width="9%">
<col width="23%">
<col width="34%">
</colgroup>
<thead><tr class="header">
<th align="left">Model</th>
<th align="left">Type</th>
<th align="left">Limits/Remarks</th>
<th align="left">Reference</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left"><code>tur99</code></td>
<td align="left">RNA</td>
<td align="left"></td>
<td align="left"><span class="citation">Mathews et al. (1999)</span></td>
</tr>
<tr class="even">
<td align="left">
<code>ser12</code>*</td>
<td align="left">RNA</td>
<td align="left"></td>
<td align="left"><span class="citation">Chen et al. (2012)</span></td>
</tr>
</tbody>
</table>
* Default method for computation.

<p><em><strong>Examples</strong></em></p>
<pre><code>    RNA:CCAGCGUCCU
        ||||||||||
    RNA:GGTCGCAGGA</code></pre>
<div class="sourceCode" id="cb90"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># GU wobble base pairs effect - default (RNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CCAGCGUCCU"</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 79.46955</code></pre>
<div class="sourceCode" id="cb92"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># GU wobble base pairs effect - ser12 (RNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CCAGCGUCCU"</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span>, method.GU <span class="op">=</span> <span class="st">"ser12"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 79.46955</code></pre>
<div class="sourceCode" id="cb94"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># GU wobble base pairs effect - tur99 (RNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CCAGCGUCCU"</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>,
        hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span>, method.GU <span class="op">=</span> <span class="st">"tur99"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 79.46955</code></pre>
</div>
<div id="single-mismatch-effect" class="section level3">
<h3 class="hasAnchor">
<a href="#single-mismatch-effect" class="anchor"></a>Single mismatch effect</h3>
<p>The nearest neighbour model for computation in case of sequences with a single mismatch can be specified by the argument <code>method.singleMM</code>. The available methods are given in <strong>Table 6</strong>.</p>
<p><strong>Table 6</strong>: Details of methods for sequences with single mismatch</p>
<table class="table">
<colgroup>
<col width="19%">
<col width="12%">
<col width="24%">
<col width="42%">
</colgroup>
<thead><tr class="header">
<th align="left">Model</th>
<th align="left">Type</th>
<th align="left">Limits/Remarks</th>
<th align="left">Reference</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">
<code>allsanpey</code>*</td>
<td align="left">DNA</td>
<td align="left"></td>
<td align="left">
<span class="citation">Allawi and SantaLucia (1997)</span>, <span class="citation">Allawi and SantaLucia (1998a)</span>, <span class="citation">Allawi and SantaLucia (1998b)</span>, <span class="citation">Allawi and SantaLucia (1998c)</span>, <span class="citation">Peyret et al. (1999)</span>
</td>
</tr>
<tr class="even">
<td align="left">
<code>wat11</code>*</td>
<td align="left">DNA/RNA</td>
<td align="left"></td>
<td align="left"><span class="citation">Watkins et al. (2011)</span></td>
</tr>
<tr class="odd">
<td align="left"><code>tur06</code></td>
<td align="left">RNA</td>
<td align="left"></td>
<td align="left"><span class="citation">Lu et al. (2006)</span></td>
</tr>
<tr class="even">
<td align="left">
<code>zno07</code>*</td>
<td align="left">RNA</td>
<td align="left"></td>
<td align="left"><span class="citation">Davis and Znosko (2007)</span></td>
</tr>
<tr class="odd">
<td align="left"><code>zno08</code></td>
<td align="left">RNA</td>
<td align="left">At least one adjacent GU base pair.</td>
<td align="left"><span class="citation">Davis and Znosko (2008)</span></td>
</tr>
</tbody>
</table>
* Default method for computation.

<p><em><strong>Examples</strong></em></p>
<pre><code>    DNA:CAACTTGATATTAATA
        |||||||| |||||||
    DNA:GTTGAACTCTAATTAT

    RNA:GACAGGCUG
        |||| ||||
    RNA:CUGUGCGAC
        
    DNA:CCATAACTACC
        |||| ||||||
    RNA:GGUAAUGAUGG</code></pre>
<div class="sourceCode" id="cb97"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Single mismatch effect - default (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAACTTGATATTAATA"</span>, comp.sequence <span class="op">=</span> <span class="st">"GTTGAACTCTAATTAT"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0004</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 51.97499</code></pre>
<div class="sourceCode" id="cb99"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Single mismatch effect - allsanpey (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAACTTGATATTAATA"</span>, comp.sequence <span class="op">=</span> <span class="st">"GTTGAACTCTAATTAT"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0004</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, method.singleMM <span class="op">=</span> <span class="st">"allsanpey"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 51.97499</code></pre>
<div class="sourceCode" id="cb101"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Single mismatch effect - default (RNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"GACAGGCUG"</span>, comp.sequence <span class="op">=</span> <span class="st">"CUGUGCGAC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 54.40363</code></pre>
<div class="sourceCode" id="cb103"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Single mismatch effect - zno07 (RNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"GACAGGCUG"</span>, comp.sequence <span class="op">=</span> <span class="st">"CUGUGCGAC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, method.singleMM <span class="op">=</span> <span class="st">"zno07"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 54.40363</code></pre>
<div class="sourceCode" id="cb105"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Single mismatch effect - zno08 (RNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAGUACGUC"</span>, comp.sequence <span class="op">=</span> <span class="st">"GUCGGGCAG"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, method.singleMM <span class="op">=</span> <span class="st">"zno08"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 38.26298</code></pre>
<div class="sourceCode" id="cb107"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Single mismatch effect - tur06 (RNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"GACAGGCUG"</span>, comp.sequence <span class="op">=</span> <span class="st">"CUGUGCGAC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, method.singleMM <span class="op">=</span> <span class="st">"tur06"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 58.27825</code></pre>
<div class="sourceCode" id="cb109"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Single mismatch effect - default (DNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CCATAACTACC"</span>, comp.sequence <span class="op">=</span> <span class="st">"GGUAAUGAUGG"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnarna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 40.32976</code></pre>
<div class="sourceCode" id="cb111"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Single mismatch effect - wat11 (DNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CCATAACTACC"</span>, comp.sequence <span class="op">=</span> <span class="st">"GGUAAUGAUGG"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnarna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, method.singleMM <span class="op">=</span> <span class="st">"wat11"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 40.32976</code></pre>
</div>
<div id="tandem-mismatches-effect" class="section level3">
<h3 class="hasAnchor">
<a href="#tandem-mismatches-effect" class="anchor"></a>Tandem mismatches effect</h3>
<p>The nearest neighbour model for computation in case of sequences with tandem mismatches can be specified by the argument <code>method.tandemMM</code>. The available methods are given in <strong>Table 7</strong>.</p>
<p><strong>Table 7</strong>: Details of methods for sequences with tandem mismatches</p>
<table class="table">
<colgroup>
<col width="19%">
<col width="8%">
<col width="29%">
<col width="42%">
</colgroup>
<thead><tr class="header">
<th align="left">Model</th>
<th align="left">Type</th>
<th align="left">Limits/Remarks</th>
<th align="left">Reference</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">
<code>allsanpey</code>*</td>
<td align="left">DNA</td>
<td align="left">Only GT mismatches and TA/TG mismatches.</td>
<td align="left">
<span class="citation">Allawi and SantaLucia (1997)</span>, <span class="citation">Allawi and SantaLucia (1998a)</span>, <span class="citation">Allawi and SantaLucia (1998b)</span>, <span class="citation">Allawi and SantaLucia (1998c)</span>, <span class="citation">Peyret et al. (1999)</span>
</td>
</tr>
<tr class="even">
<td align="left">
<code>tur99</code>*</td>
<td align="left">RNA</td>
<td align="left">No adjacent GU or UG base pairs.</td>
<td align="left">
<span class="citation">Mathews et al. (1999)</span>, <span class="citation">Lu et al. (2006)</span>
</td>
</tr>
</tbody>
</table>
* Default method for computation.

<p><em><strong>Examples</strong></em></p>
<pre><code>    DNA:GACGTTGGAC
        ||||  ||||
    DNA:CTGCGGCCTG

    RNA:GAGCGGAG
        |||  |||
    RNA:CUCCACUC</code></pre>
<div class="sourceCode" id="cb114"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Tandem mismatches effect - default (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"GACGTTGGAC"</span>, comp.sequence <span class="op">=</span> <span class="st">"CTGCGGCCTG"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0004</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 50.20175</code></pre>
<div class="sourceCode" id="cb116"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Tandem mismatches effect - allsanpey (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"GACGTTGGAC"</span>, comp.sequence <span class="op">=</span> <span class="st">"CTGCGGCCTG"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0004</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, method.tandemMM <span class="op">=</span> <span class="st">"allsanpey"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 50.20175</code></pre>
<div class="sourceCode" id="cb118"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Tandem mismatches effect - default (RNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"GAGCGGAG"</span>, comp.sequence <span class="op">=</span> <span class="st">"CUCCACUC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 21.07224</code></pre>
<div class="sourceCode" id="cb120"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Tandem mismatches effect -  tur06 (RNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"GAGCGGAG"</span>, comp.sequence <span class="op">=</span> <span class="st">"CUCCACUC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, method.tandemMM <span class="op">=</span> <span class="st">"tur99"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 21.07224</code></pre>
</div>
<div id="single-dangling-end-effect" class="section level3">
<h3 class="hasAnchor">
<a href="#single-dangling-end-effect" class="anchor"></a>Single dangling end effect</h3>
<p>The nearest neighbour model for computation in case of sequences with a single dangling end can be specified by the argument <code>method.single.dangle</code>. The available methods are given in <strong>Table 8</strong>.</p>
<p><strong>Table 8</strong>: Details of methods for sequences with single dangling end</p>
<table class="table">
<colgroup>
<col width="17%">
<col width="9%">
<col width="30%">
<col width="42%">
</colgroup>
<thead><tr class="header">
<th align="left">Model</th>
<th align="left">Type</th>
<th align="left">Limits/Remarks</th>
<th align="left">Reference</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">
<code>bom00</code>*</td>
<td align="left">DNA</td>
<td align="left"></td>
<td align="left"><span class="citation">Bommarito et al. (2000)</span></td>
</tr>
<tr class="even">
<td align="left"><code>sugdna02</code></td>
<td align="left">DNA</td>
<td align="left">Only terminal poly A self complementary sequences.</td>
<td align="left"><span class="citation">Ohmichi et al. (2002)</span></td>
</tr>
<tr class="odd">
<td align="left"><code>sugrna02</code></td>
<td align="left">RNA</td>
<td align="left">Only terminal poly A self complementary sequences.</td>
<td align="left"><span class="citation">Ohmichi et al. (2002)</span></td>
</tr>
<tr class="even">
<td align="left">
<code>ser08</code>*</td>
<td align="left">RNA</td>
<td align="left">Only 3’ UA, GU and UG terminal base pairs only 5’ UG and GU terminal base pairs.</td>
<td align="left">
<span class="citation">O’Toole et al. (2006)</span>, <span class="citation">Miller et al. (2008)</span>
</td>
</tr>
</tbody>
</table>
* Default method for computation.

<p><em><strong>Examples</strong></em></p>
<pre><code>    DNA:-GTAGCTACA
         |||||||| 
    DNA:ACATCGATG-

    RNA:-GGCGCUG
         |||||||
    RNA: CCGCGAC
        
    DNA:-GGCGCUG
         |||||||
    RNA: CCGCGAC</code></pre>
<div class="sourceCode" id="cb123"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Single dangling end effect - default (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"-GTAGCTACA"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0004</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 52.58935</code></pre>
<div class="sourceCode" id="cb125"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Single dangling end effect - bom00 (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"-GTAGCTACA"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0004</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, method.single.dangle <span class="op">=</span> <span class="st">"bom00"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 52.58935</code></pre>
<div class="sourceCode" id="cb127"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Single dangling end effect - sugdna02 (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"-GTAGCTACA"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0004</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, method.single.dangle <span class="op">=</span> <span class="st">"sugdna02"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 50.78548</code></pre>
<div class="sourceCode" id="cb129"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Single dangling end effect - default (RNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"-GGCGCUG"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 65.7647</code></pre>
<div class="sourceCode" id="cb131"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Single dangling end effect -  ser08 (RNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"-GGCGCUG"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, method.single.dangle <span class="op">=</span> <span class="st">"ser08"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 65.7647</code></pre>
<div class="sourceCode" id="cb133"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Single dangling end effect -  sugrna02 (RNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"-GGCGCUG"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, method.single.dangle <span class="op">=</span> <span class="st">"sugrna02"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 65.7647</code></pre>
</div>
<div id="double-dangling-end-effect" class="section level3">
<h3 class="hasAnchor">
<a href="#double-dangling-end-effect" class="anchor"></a>Double dangling end effect</h3>
<p>The nearest neighbour model for computation in case of sequences with a double or secondary dangling ends can be specified by the argument <code>method.double.dangle</code>. The available methods are given in <strong>Table 9</strong>.</p>
<p><strong>Table 9</strong>: Details of methods for sequences with double dangling ends</p>
<table class="table">
<colgroup>
<col width="19%">
<col width="9%">
<col width="31%">
<col width="39%">
</colgroup>
<thead><tr class="header">
<th align="left">Model</th>
<th align="left">Type</th>
<th align="left">Limits/Remarks</th>
<th align="left">Reference</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">
<code>sugdna02</code>*</td>
<td align="left">DNA</td>
<td align="left">Only terminal poly A self complementary sequences.</td>
<td align="left"><span class="citation">Ohmichi et al. (2002)</span></td>
</tr>
<tr class="even">
<td align="left"><code>sugrna02</code></td>
<td align="left">RNA</td>
<td align="left">Only terminal poly A self complementary sequences.</td>
<td align="left"><span class="citation">Ohmichi et al. (2002)</span></td>
</tr>
<tr class="odd">
<td align="left"><code>ser05</code></td>
<td align="left">RNA</td>
<td align="left">Depends on the available thermodynamic parameters for single dangling end.</td>
<td align="left"><span class="citation">O’Toole et al. (2005)</span></td>
</tr>
<tr class="even">
<td align="left">
<code>ser06</code>*</td>
<td align="left">RNA</td>
<td align="left"></td>
<td align="left"><span class="citation">O’Toole et al. (2006)</span></td>
</tr>
</tbody>
</table>
* Default method for computation.

<p><em><strong>Examples</strong></em></p>
<pre><code>    DNA:--ATGCATAA
          ||||||
    DNA:AATACGTA--

    RNA:--AUGCAUAA
          ||||||
    RNA:AAUACGUA--</code></pre>
<div class="sourceCode" id="cb136"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Double dangling end effect - default (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"--ATGCATAA"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0004</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 44.88615</code></pre>
<div class="sourceCode" id="cb138"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Double dangling end effect - sugdna02 (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"--ATGCATAA"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0004</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, method.double.dangle <span class="op">=</span> <span class="st">"sugdna02"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 44.88615</code></pre>
<div class="sourceCode" id="cb140"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Double dangling end effect - default (RNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"--AUGCAUAA"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 42.79724</code></pre>
<div class="sourceCode" id="cb142"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Double dangling end effect -  ser06 (RNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"--AUGCAUAA"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, method.double.dangle <span class="op">=</span> <span class="st">"ser06"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 42.79724</code></pre>
<div class="sourceCode" id="cb144"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Double dangling end effect -  sugrna02 (RNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"--AUGCAUAA"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, method.double.dangle <span class="op">=</span> <span class="st">"sugrna02"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 41.82788</code></pre>
<div class="sourceCode" id="cb146"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Double dangling end effect -  ser05 (RNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"--AUGCAUAA"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, method.double.dangle <span class="op">=</span> <span class="st">"ser05"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 42.78815</code></pre>
</div>
<div id="long-dangling-end-effect" class="section level3">
<h3 class="hasAnchor">
<a href="#long-dangling-end-effect" class="anchor"></a>Long dangling end effect</h3>
<p>The nearest neighbour model for computation in case of sequences with a double or secondary dangling ends can be specified by the argument <code>method.long.dangle</code>. The available methods are given in <strong>Table 10</strong>.</p>
<p><strong>Table 10</strong>: Details of methods for sequences with long dangling ends</p>
<table class="table">
<colgroup>
<col width="19%">
<col width="9%">
<col width="31%">
<col width="31%">
</colgroup>
<thead><tr class="header">
<th align="left">Model</th>
<th align="left">Type</th>
<th align="left">Limits/Remarks</th>
<th align="left">Reference</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">
<code>sugdna02</code>*</td>
<td align="left">DNA</td>
<td align="left">Only terminal poly A self complementary sequences.</td>
<td align="left"><span class="citation">Ohmichi et al. (2002)</span></td>
</tr>
<tr class="even">
<td align="left">
<code>sugrna02</code>*</td>
<td align="left">RNA</td>
<td align="left">Only terminal poly A self complementary sequences.</td>
<td align="left"><span class="citation">Ohmichi et al. (2002)</span></td>
</tr>
</tbody>
</table>
* Default method for computation.

<p><em><strong>Examples</strong></em></p>
<pre><code>    DNA:----GCATATGCAAAA
            |||||||| 
    DNA:AAAACGTATACG----

    RNA:AAAAGCAUAUGC----
            ||||||||
    RNA:----CGUAUACGAAAA</code></pre>
<div class="sourceCode" id="cb149"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Long dangling end effect - default (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"----GCATATGCAAAA"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0004</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 55.69854</code></pre>
<div class="sourceCode" id="cb151"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Long dangling end effect - sugdna02 (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"----GCATATGCAAAA"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0004</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, method.long.dangle <span class="op">=</span> <span class="st">"sugdna02"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 55.69854</code></pre>
<div class="sourceCode" id="cb153"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Long dangling end effect - default (RNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"AAAAGCAUAUGC----"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 57.21314</code></pre>
<div class="sourceCode" id="cb155"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Long dangling end effect -  sugrna02 (RNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"AAAAGCAUAUGC----"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, method.long.dangle <span class="op">=</span> <span class="st">"sugrna02"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 57.21314</code></pre>
</div>
<div id="internal-loop-effect" class="section level3">
<h3 class="hasAnchor">
<a href="#internal-loop-effect" class="anchor"></a>Internal loop effect</h3>
<p>The nearest neighbour model for computation in case of sequences with an internal loop (more than two adjacent mismatches) can be specified by the argument <code>method.internal.loop</code>. The available methods are given in <strong>Table 11</strong>.</p>
<p><strong>Table 11</strong>: Details of methods for sequences with internal loops</p>
<table class="table">
<colgroup>
<col width="14%">
<col width="9%">
<col width="29%">
<col width="45%">
</colgroup>
<thead><tr class="header">
<th align="left">Model</th>
<th align="left">Type</th>
<th align="left">Limits/Remarks</th>
<th align="left">Reference</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">
<code>san04</code>*</td>
<td align="left">DNA</td>
<td align="left">Missing asymmetry penalty. Not tested with experimental results.</td>
<td align="left"><span class="citation">SantaLucia and Hicks (2004)</span></td>
</tr>
<tr class="even">
<td align="left"><code>tur06</code></td>
<td align="left">RNA</td>
<td align="left">Not tested with experimental results.</td>
<td align="left"><span class="citation">Lu et al. (2006)</span></td>
</tr>
<tr class="odd">
<td align="left">
<code>zno07</code>*</td>
<td align="left">RNA</td>
<td align="left">Only for 1x2 loop.</td>
<td align="left"><span class="citation">Badhwar et al. (2007)</span></td>
</tr>
</tbody>
</table>
* Default method for computation.

<p><em><strong>Examples</strong></em></p>
<pre><code>    DNA:GCGATTGGCACTTTGGTGAAC
        |||||    ||||||||||||
    DNA:CGCTACATATGAAACCACTTG

    RNA:GACAC-GCUG
        ||||  ||||
    RNA:CUGUAUCGAC</code></pre>
<div class="sourceCode" id="cb158"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Internal loop effect - default (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"GCGATTGGCACTTTGGTGAAC"</span>, comp.sequence <span class="op">=</span> <span class="st">"CGCTACATATGAAACCACTTG"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 84.09052</code></pre>
<div class="sourceCode" id="cb160"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Internal loop effect - san04 (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"GCGATTGGCACTTTGGTGAAC"</span>, comp.sequence <span class="op">=</span> <span class="st">"CGCTACATATGAAACCACTTG"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, method.internal.loop <span class="op">=</span> <span class="st">"san04"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 84.09052</code></pre>
<div class="sourceCode" id="cb162"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Internal loop effect - default (RNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"GACAC-GCUG"</span>, comp.sequence <span class="op">=</span> <span class="st">"CUGUAUCGAC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 45.98713</code></pre>
<div class="sourceCode" id="cb164"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Internal loop effect -  zno07 (RNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"GACAC-GCUG"</span>, comp.sequence <span class="op">=</span> <span class="st">"CUGUAUCGAC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, method.internal.loop <span class="op">=</span> <span class="st">"zno07"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 40.49012</code></pre>
<div class="sourceCode" id="cb166"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Internal loop effect -  tur06 (RNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"GACAC-GCUG"</span>, comp.sequence <span class="op">=</span> <span class="st">"CUGUAUCGAC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, method.internal.loop <span class="op">=</span> <span class="st">"tur06"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 45.98713</code></pre>
</div>
<div id="single-bulge-loop-effect" class="section level3">
<h3 class="hasAnchor">
<a href="#single-bulge-loop-effect" class="anchor"></a>Single bulge loop effect</h3>
<p>The nearest neighbour model for computation in case of sequences with a single bulge loop can be specified by the argument <code>method.single.bulge.loop</code>. The available methods are given in <strong>Table 12</strong>.</p>
<p><strong>Table 12</strong>: Details of methods for sequences with single bulge loop</p>
<table class="table">
<colgroup>
<col width="14%">
<col width="9%">
<col width="29%">
<col width="46%">
</colgroup>
<thead><tr class="header">
<th align="left">Model</th>
<th align="left">Type</th>
<th align="left">Limits/Remarks</th>
<th align="left">Reference</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">
<code>tan04</code>*</td>
<td align="left">DNA</td>
<td align="left"></td>
<td align="left"><span class="citation">Tan and Chen (2007)</span></td>
</tr>
<tr class="even">
<td align="left"><code>san04</code></td>
<td align="left">DNA</td>
<td align="left">Missing closing AT penalty.</td>
<td align="left"><span class="citation">SantaLucia and Hicks (2004)</span></td>
</tr>
<tr class="odd">
<td align="left"><code>ser07</code></td>
<td align="left">RNA</td>
<td align="left">Less reliable results. Some missing parameters.</td>
<td align="left"><span class="citation">Blose et al. (2007)</span></td>
</tr>
<tr class="even">
<td align="left">
<code>tur06</code>*</td>
<td align="left">RNA</td>
<td align="left"></td>
<td align="left"><span class="citation">Lu et al. (2006)</span></td>
</tr>
</tbody>
</table>
* Default method for computation.

<p><em><strong>Examples</strong></em></p>
<pre><code>    DNA:TCGATTAGCGACACAGG
        |||||||| ||||||||
    DNA:AGCTAATC-CTGTGTCC

    RNA:GACUCUGUC
        |||| ||||
    RNA:CUGA-ACAG</code></pre>
<div class="sourceCode" id="cb169"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Single bulge loop effect - default (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"TCGATTAGCGACACAGG"</span>, comp.sequence <span class="op">=</span> <span class="st">"AGCTAATC-CTGTGTCC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 71.12754</code></pre>
<div class="sourceCode" id="cb171"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Single bulge loop effect - tan04 (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"TCGATTAGCGACACAGG"</span>, comp.sequence <span class="op">=</span> <span class="st">"AGCTAATC-CTGTGTCC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, method.single.bulge.loop <span class="op">=</span> <span class="st">"tan04"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 71.12754</code></pre>
<div class="sourceCode" id="cb173"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Single bulge loop effect - san04 (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"TCGATTAGCGACACAGG"</span>, comp.sequence <span class="op">=</span> <span class="st">"AGCTAATC-CTGTGTCC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, method.single.bulge.loop <span class="op">=</span> <span class="st">"san04"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 62.0496</code></pre>
<div class="sourceCode" id="cb175"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Single bulge loop effect - default (RNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"GACUCUGUC"</span>, comp.sequence <span class="op">=</span> <span class="st">"CUGA-ACAG"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 39.47787</code></pre>
<div class="sourceCode" id="cb177"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Single bulge loop effect -  tur06 (RNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"GACUCUGUC"</span>, comp.sequence <span class="op">=</span> <span class="st">"CUGA-ACAG"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, method.single.bulge.loop <span class="op">=</span> <span class="st">"tur06"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 39.47787</code></pre>
<div class="sourceCode" id="cb179"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Single bulge loop effect -  ser07 (RNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"GACUCUGUC"</span>, comp.sequence <span class="op">=</span> <span class="st">"CUGA-ACAG"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, method.single.bulge.loop <span class="op">=</span> <span class="st">"ser07"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 31.42849</code></pre>
</div>
<div id="long-bulge-loop-effect" class="section level3">
<h3 class="hasAnchor">
<a href="#long-bulge-loop-effect" class="anchor"></a>Long bulge loop effect</h3>
<p>The nearest neighbour model for computation in case of sequences with long bulge loop can be specified by the argument <code>method.long.bulge.loop</code>. The available methods are given in <strong>Table 13</strong>.</p>
<p><strong>Table 13</strong>: Details of methods for sequences with long bulge loop</p>
<table class="table">
<colgroup>
<col width="15%">
<col width="9%">
<col width="28%">
<col width="46%">
</colgroup>
<thead><tr class="header">
<th align="left">Model</th>
<th align="left">Type</th>
<th align="left">Limits/Remarks</th>
<th align="left">Reference</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">
<code>san04</code>*</td>
<td align="left">DNA</td>
<td align="left">Missing closing AT penalty.</td>
<td align="left"><span class="citation">SantaLucia and Hicks (2004)</span></td>
</tr>
<tr class="even">
<td align="left">
<code>tur06</code>*</td>
<td align="left">RNA</td>
<td align="left">Not tested with experimental results.</td>
<td align="left">
<span class="citation">Mathews et al. (1999)</span>, <span class="citation">Lu et al. (2006)</span>
</td>
</tr>
</tbody>
</table>
* Default method for computation.

<p><em><strong>Examples</strong></em></p>
<pre><code>    DNA:ATATGACGCCACAGCG
        |||||   ||||||||
    DNA:TATAC---GGTGTCGC

    RNA:AUAUGACGCCACAGCG
        |||||   ||||||||
    RNA:UAUAC---GGUGUCGC</code></pre>
<div class="sourceCode" id="cb182"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Long bulge loop effect - default (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"ATATGACGCCACAGCG"</span>, comp.sequence <span class="op">=</span> <span class="st">"TATAC---GGTGTCGC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 51.7104</code></pre>
<div class="sourceCode" id="cb184"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Long bulge loop effect - san04 (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"ATATGACGCCACAGCG"</span>, comp.sequence <span class="op">=</span> <span class="st">"TATAC---GGTGTCGC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, method.long.bulge.loop <span class="op">=</span> <span class="st">"san04"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 51.7104</code></pre>
<div class="sourceCode" id="cb186"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Long bulge loop effect - default (RNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"AUAUGACGCCACAGCG"</span>, comp.sequence <span class="op">=</span> <span class="st">"UAUAC---GGUGUCGC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 66.0497</code></pre>
<div class="sourceCode" id="cb188"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Long bulge loop effect -  tur06 (RNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"AUAUGACGCCACAGCG"</span>, comp.sequence <span class="op">=</span> <span class="st">"UAUAC---GGUGUCGC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, method.long.bulge.loop <span class="op">=</span> <span class="st">"tur06"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 66.0497</code></pre>
</div>
<div id="cng-repeats-effect" class="section level3">
<h3 class="hasAnchor">
<a href="#cng-repeats-effect" class="anchor"></a>CNG repeats effect</h3>
<p>The nearest neighbour model for computation in case of sequences with CNG repeats can be specified by the argument <code>method.CNG</code>. The available methods are given in <strong>Table 14</strong>.</p>
<p><strong>Table 14</strong>: Details of methods for sequences with CNG repeats</p>
<table class="table">
<colgroup>
<col width="15%">
<col width="9%">
<col width="29%">
<col width="38%">
</colgroup>
<thead><tr class="header">
<th align="left">Model</th>
<th align="left">Type</th>
<th align="left">Limits/Remarks</th>
<th align="left">Reference</th>
</tr></thead>
<tbody><tr class="odd">
<td align="left">
<code>bro05</code>*</td>
<td align="left">RNA</td>
<td align="left">Self complementary sequences. 2 to 7 CNG repeats.</td>
<td align="left"><span class="citation">Broda et al. (2005)</span></td>
</tr></tbody>
</table>
* Default method for computation.

<p><em><strong>Examples</strong></em></p>
<pre><code>    RNA:GCGGCGGCGGC
        |||||||||||
    RNA:CGCCGCCGCCG</code></pre>
<div class="sourceCode" id="cb191"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># CNG repeats effect - default (RNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"GCGGCGGCGGC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 94.25719</code></pre>
<div class="sourceCode" id="cb193"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># CNG repeats effect -  bro05 (RNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"GCGGCGGCGGC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, method.CNG <span class="op">=</span> <span class="st">"bro05"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 94.25719</code></pre>
</div>
<div id="inosine-bases-effect" class="section level3">
<h3 class="hasAnchor">
<a href="#inosine-bases-effect" class="anchor"></a>Inosine bases effect</h3>
<p>The nearest neighbour model for computation in case of sequences with inosine bases (I) can be specified by the argument <code>method.inosine</code>. The available methods are given in <strong>Table 15</strong>.</p>
<p><strong>Table 15</strong>: Details of methods for sequences with inosine bases</p>
<table class="table">
<colgroup>
<col width="15%">
<col width="9%">
<col width="30%">
<col width="45%">
</colgroup>
<thead><tr class="header">
<th align="left">Model</th>
<th align="left">Type</th>
<th align="left">Limits/Remarks</th>
<th align="left">Reference</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">
<code>san05</code>*</td>
<td align="left">DNA</td>
<td align="left">Missing parameters for tandem base pairs containing inosine bases.</td>
<td align="left"><span class="citation">Watkins and SantaLucia (2005)</span></td>
</tr>
<tr class="even">
<td align="left">
<code>zno07</code>*</td>
<td align="left">RNA</td>
<td align="left">Only IU base pairs.</td>
<td align="left"><span class="citation">Wright et al. (2007)</span></td>
</tr>
</tbody>
</table>
* Default method for computation.

<p><em><strong>Examples</strong></em></p>
<pre><code>    DNA:CCGICTGTIGCG
        ||| |||| |||
    DNA:GGCCGACACCGC

    RNA:GCAICGC
        ||| |||
    RNA:CGUUGCG</code></pre>
<div class="sourceCode" id="cb196"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Inosine bases effect - default (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CCGICTGTIGCG"</span>, comp.sequence <span class="op">=</span> <span class="st">"GGCCGACACCGC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 65.36853</code></pre>
<div class="sourceCode" id="cb198"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Inosine bases effect - san05 (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CCGICTGTIGCG"</span>, comp.sequence <span class="op">=</span> <span class="st">"GGCCGACACCGC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, method.inosine <span class="op">=</span> <span class="st">"san05"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 65.36853</code></pre>
<div class="sourceCode" id="cb200"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Inosine bases effect - default (RNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"GCAICGC"</span>, comp.sequence <span class="op">=</span> <span class="st">"CGUUGCG"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 46.75042</code></pre>
<div class="sourceCode" id="cb202"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Inosine bases effect -  zno07 (RNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"GCAICGC"</span>, comp.sequence <span class="op">=</span> <span class="st">"CGUUGCG"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, method.inosine <span class="op">=</span> <span class="st">"zno07"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 46.75042</code></pre>
</div>
<div id="hydroxyadenine-bases-effect" class="section level3">
<h3 class="hasAnchor">
<a href="#hydroxyadenine-bases-effect" class="anchor"></a>Hydroxyadenine bases effect</h3>
<p>The nearest neighbour model for computation in case of sequences with hydroxyadenine bases can be specified by the argument <code>method.hydroxyadenine</code>. The available methods are given in <strong>Table 16</strong>.</p>
<p><strong>Table 16</strong>: Details of methods for sequences with hydroxyadenine bases</p>
<table class="table">
<colgroup>
<col width="15%">
<col width="9%">
<col width="30%">
<col width="43%">
</colgroup>
<thead><tr class="header">
<th align="left">Model</th>
<th align="left">Type</th>
<th align="left">Limits/Remarks</th>
<th align="left">Reference</th>
</tr></thead>
<tbody><tr class="odd">
<td align="left">
<code>sug01</code>*</td>
<td align="left">DNA</td>
<td align="left">Only 5’ GA*C 3’and 5’ TA*A 3’ contexts.</td>
<td align="left"><span class="citation">Kawakami et al. (2001)</span></td>
</tr></tbody>
</table>
* Default method for computation.

<p><em><strong>Examples</strong></em></p>
<pre><code>               *
    DNA:AGAAATGACACGGTG
        |||||||||||||||
    DNA:TCTTTACCGTGCCAC</code></pre>
<div class="sourceCode" id="cb205"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Hydroxyadenine bases effect - default (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"AGAAATGA*CACGGTG"</span>, comp.sequence <span class="op">=</span> <span class="st">"TCTTTACCGTGCCAC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 68.46041</code></pre>
<div class="sourceCode" id="cb207"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Hydroxyadenine bases effect - sug01 (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"AGAAATGA*CACGGTG"</span>, comp.sequence <span class="op">=</span> <span class="st">"TCTTTACCGTGCCAC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, method.hydroxyadenine <span class="op">=</span> <span class="st">"sug01"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 68.46041</code></pre>
</div>
<div id="azobenzenes-effect" class="section level3">
<h3 class="hasAnchor">
<a href="#azobenzenes-effect" class="anchor"></a>Azobenzenes effect</h3>
<p>The nearest neighbour model for computation in case of sequences with azobenzenes (X_T for trans azobenzenes and X_C for cis azobenzenes) can be specified by the argument <code>method.azobenzenes</code>. The available methods are given in <strong>Table 17</strong>.</p>
<p><strong>Table 17</strong>: Details of methods for sequences with azobenzenes</p>
<table class="table">
<colgroup>
<col width="15%">
<col width="9%">
<col width="26%">
<col width="36%">
</colgroup>
<thead><tr class="header">
<th align="left">Model</th>
<th align="left">Type</th>
<th align="left">Limits/Remarks</th>
<th align="left">Reference</th>
</tr></thead>
<tbody><tr class="odd">
<td align="left">
<code>asa05</code>*</td>
<td align="left">DNA</td>
<td align="left">Less reliable results when the number of cis azobenzene increases.</td>
<td align="left"><span class="citation">Asanuma et al. (2005)</span></td>
</tr></tbody>
</table>
* Default method for computation.

<p><em><strong>Examples</strong></em></p>
<pre><code>          C    C    C    C    C
    DNA:CTXTTAAXGAAGXGAGAXTATAXCC
        || |||| |||| |||| |||| ||
    DNA:GA AATT CTTC CTCT ATAT GG</code></pre>
<div class="sourceCode" id="cb210"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Azobenzenes effect - default (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CTX_CTTAAX_CGAAGX_CGAGAX_CTATAX_CCC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 47.85385</code></pre>
<div class="sourceCode" id="cb212"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Azobenzenes effect - asa05 (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CTX_CTTAAX_CGAAGX_CGAGAX_CTATAX_CCC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, method.azobenzenes <span class="op">=</span> <span class="st">"asa05"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 47.85385</code></pre>
</div>
<div id="locked-nucleic-acids-effect" class="section level3">
<h3 class="hasAnchor">
<a href="#locked-nucleic-acids-effect" class="anchor"></a>Locked nucleic acids effect</h3>
<p>The nearest neighbour model for computation in case of sequences with locked nucleic acids can be specified by the argument <code>method.locked</code>. The available methods are given in <strong>Table 18</strong>.</p>
<p><strong>Table 18</strong>: Details of methods for sequences with locked nucleic acids</p>
<table class="table">
<colgroup>
<col width="15%">
<col width="9%">
<col width="23%">
<col width="48%">
</colgroup>
<thead><tr class="header">
<th align="left">Model</th>
<th align="left">Type</th>
<th align="left">Limits/Remarks</th>
<th align="left">Reference</th>
</tr></thead>
<tbody><tr class="odd">
<td align="left">
<code>mct04</code>*</td>
<td align="left">DNA</td>
<td align="left"></td>
<td align="left"><span class="citation">McTigue et al. (2004)</span></td>
</tr></tbody>
</table>
* Default method for computation.

<p><em><strong>Examples</strong></em></p>
<pre><code>             L  
    DNA:CCATTGCTACC
        |||||||||||
    DNA:GGTAACGATGG</code></pre>
<div class="sourceCode" id="cb215"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Locked nucleic acids effect - default (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CCATTLGCTACC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 63.61426</code></pre>
<div class="sourceCode" id="cb217"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Locked nucleic acids effect - sug01 (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CCATTLGCTACC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0001</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, method.locked <span class="op">=</span> <span class="st">"mct04"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 63.61426</code></pre>
</div>
</div>
</div>
<div id="corrections" class="section level1">
<h1 class="hasAnchor">
<a href="#corrections" class="anchor"></a>Corrections</h1>
<p>Once the melting temperature is computed, a correction is applied to it according to the concentration of nucleic acids, cations and/or denaturing agents.</p>
<div id="nucleic-acid-concentration" class="section level2">
<h2 class="hasAnchor">
<a href="#nucleic-acid-concentration" class="anchor"></a>Nucleic acid concentration</h2>
<p>For self complementary sequences (auto detected or specified by <code>force.self</code>) it is 1. Otherwise it is 4 if the both strands are present in equivalent amount and 1 if one strand is in excess.</p>
</div>
<div id="ion-corrections" class="section level2">
<h2 class="hasAnchor">
<a href="#ion-corrections" class="anchor"></a>Ion corrections</h2>
<p>Melting temperature is computed initially for <span class="math inline">\([\textrm{Na}^{+}]\)</span> = 1 M, after which a correction for the presence of cations (<span class="math inline">\([\textrm{Na}^{+}]\)</span>, <span class="math inline">\([\textrm{K}^{+}]\)</span>, <span class="math inline">\([\textrm{Tris}^{+}]\)</span> and <span class="math inline">\([\textrm{Mg}^{+}]\)</span>) is applied either directly on the computed melting temperature or on the computed entropy.</p>
<p>Th correction methods for cation concentration can be specified by the argument <code>correction.ion</code>.</p>
<div id="sodium-corrections" class="section level3">
<h3 class="hasAnchor">
<a href="#sodium-corrections" class="anchor"></a>Sodium corrections</h3>
<p>The available correction methods for sodium concentration are given in <strong>Table 19</strong>.</p>
<p><strong>Table 19</strong>: Details of the corrections for sodium concentration</p>
<table class="table">
<colgroup>
<col width="14%">
<col width="17%">
<col width="24%">
<col width="43%">
</colgroup>
<thead><tr class="header">
<th align="left">Correction</th>
<th align="left">Type</th>
<th align="left">Limits/Remarks</th>
<th align="left">Reference</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left"><code>ahs01</code></td>
<td align="left">DNA</td>
<td align="left">Na&gt;0.</td>
<td align="left"><span class="citation">Ahsen et al. (2001)</span></td>
</tr>
<tr class="even">
<td align="left"><code>kam71</code></td>
<td align="left">DNA</td>
<td align="left">Na&gt;0; Na&gt;=0.069; Na&lt;=1.02.</td>
<td align="left"><span class="citation">Frank-Kamenetskii (1971)</span></td>
</tr>
<tr class="odd">
<td align="left"><code>marschdot</code></td>
<td align="left">DNA</td>
<td align="left">Na&gt;=0.069; Na&lt;=1.02.</td>
<td align="left">
<span class="citation">Marmur and Doty (1962)</span>, <span class="citation">Blake and Delcourt (1998)</span>
</td>
</tr>
<tr class="even">
<td align="left"><code>owc1904</code></td>
<td align="left">DNA</td>
<td align="left">Na&gt;0. (equation 19)</td>
<td align="left"><span class="citation">Owczarzy et al. (2004)</span></td>
</tr>
<tr class="odd">
<td align="left"><code>owc2004</code></td>
<td align="left">DNA</td>
<td align="left">Na&gt;0. (equation 20)</td>
<td align="left"><span class="citation">Owczarzy et al. (2004)</span></td>
</tr>
<tr class="even">
<td align="left"><code>owc2104</code></td>
<td align="left">DNA</td>
<td align="left">Na&gt;0. (equation 21)</td>
<td align="left"><span class="citation">Owczarzy et al. (2004)</span></td>
</tr>
<tr class="odd">
<td align="left">
<code>owc2204</code>*</td>
<td align="left">DNA</td>
<td align="left">Na&gt;0. (equation 22)</td>
<td align="left"><span class="citation">Owczarzy et al. (2004)</span></td>
</tr>
<tr class="even">
<td align="left"><code>san96</code></td>
<td align="left">DNA</td>
<td align="left">Na&gt;=0.1.</td>
<td align="left"><span class="citation">SantaLucia et al. (1996)</span></td>
</tr>
<tr class="odd">
<td align="left"><code>san04</code></td>
<td align="left">DNA</td>
<td align="left">Na&gt;=0.05; Na&lt;=1.1; Oligonucleotides inferior to 16 bases.</td>
<td align="left">
<span class="citation">SantaLucia and Hicks (2004)</span>, <span class="citation">SantaLucia (1998)</span>
</td>
</tr>
<tr class="even">
<td align="left"><code>schlif</code></td>
<td align="left">DNA</td>
<td align="left">Na&gt;=0.07; Na&lt;=0.12.</td>
<td align="left"><span class="citation">Schildkraut and Lifson (1965)</span></td>
</tr>
<tr class="odd">
<td align="left"><code>tanna06</code></td>
<td align="left">DNA</td>
<td align="left">Na&gt;=0.001; Na&lt;=1.</td>
<td align="left"><span class="citation">Tan and Chen (2006)</span></td>
</tr>
<tr class="even">
<td align="left">
<code>tanna07</code>*</td>
<td align="left">RNA or 2’-O-MeRNA/RNA</td>
<td align="left">Na&gt;=0.003; Na&lt;=1.</td>
<td align="left"><span class="citation">Tan and Chen (2007)</span></td>
</tr>
<tr class="odd">
<td align="left"><code>wet91</code></td>
<td align="left">RNA, DNA and RNA/DNA</td>
<td align="left">Na&gt;0.</td>
<td align="left"><span class="citation">Wetmur (1991)</span></td>
</tr>
</tbody>
</table>
* Default method for computation.

<div class="sourceCode" id="cb219"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Na correction - default (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CCAGCCAGTCTCTCC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">0.069</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 56.70492</code></pre>
<div class="sourceCode" id="cb221"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Na correction - owc2204 (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CCAGCCAGTCTCTCC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">0.069</span>, correction.ion <span class="op">=</span> <span class="st">"owc2204"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 56.70492</code></pre>
<div class="sourceCode" id="cb223"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Na correction - ahs01 (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CCAGCCAGTCTCTCC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">0.069</span>, correction.ion <span class="op">=</span> <span class="st">"ahs01"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 54.1569</code></pre>
<div class="sourceCode" id="cb225"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Na correction - kam71 (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CCAGCCAGTCTCTCC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">0.069</span>, correction.ion <span class="op">=</span> <span class="st">"kam71"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 51.72963</code></pre>
<div class="sourceCode" id="cb227"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Na correction - marschdot (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CCAGCCAGTCTCTCC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">0.069</span>, correction.ion <span class="op">=</span> <span class="st">"marschdot"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 49.18075</code></pre>
<div class="sourceCode" id="cb229"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Na correction - owc1904 (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CCAGCCAGTCTCTCC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">0.069</span>, correction.ion <span class="op">=</span> <span class="st">"owc1904"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 56.18571</code></pre>
<div class="sourceCode" id="cb231"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Na correction - owc2004 (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CCAGCCAGTCTCTCC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">0.069</span>, correction.ion <span class="op">=</span> <span class="st">"owc2004"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 56.67553</code></pre>
<div class="sourceCode" id="cb233"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Na correction - owc2104 (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CCAGCCAGTCTCTCC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">0.069</span>, correction.ion <span class="op">=</span> <span class="st">"owc2104"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 56.63967</code></pre>
<div class="sourceCode" id="cb235"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Na correction - san96 (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CCAGCCAGTCTCTCC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">0.069</span>, correction.ion <span class="op">=</span> <span class="st">"san96"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 53.01651</code></pre>
<div class="sourceCode" id="cb237"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Na correction - san04 (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CCAGCCAGTCTCTCC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">0.069</span>, correction.ion <span class="op">=</span> <span class="st">"san04"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 54.15157</code></pre>
<div class="sourceCode" id="cb239"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Na correction - schlif (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CCAGCCAGTCTCTCC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">0.069</span>, correction.ion <span class="op">=</span> <span class="st">"schlif"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 48.25579</code></pre>
<div class="sourceCode" id="cb241"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Na correction - tanna06 (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CCAGCCAGTCTCTCC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">0.069</span>, correction.ion <span class="op">=</span> <span class="st">"tanna06"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 55.26711</code></pre>
<div class="sourceCode" id="cb243"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Na correction - wet91 (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CCAGCCAGTCTCTCC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">0.069</span>, correction.ion <span class="op">=</span> <span class="st">"wet91"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 51.74573</code></pre>
<div class="sourceCode" id="cb245"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Na correction - default (RNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CCAGCCAGUCUCUCC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">0.069</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 75.1552</code></pre>
<div class="sourceCode" id="cb247"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Na correction -  tanna07 (RNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CCAGCCAGUCUCUCC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">0.069</span>, correction.ion <span class="op">=</span> <span class="st">"tanna07"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 75.1552</code></pre>
<div class="sourceCode" id="cb249"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Na correction -  wet91 (RNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CCAGCCAGUCUCUCC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">0.069</span>, correction.ion <span class="op">=</span> <span class="st">"wet91"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 69.55572</code></pre>
<div class="sourceCode" id="cb251"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Na correction - default (mRNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"UACGCGUCAAUAACGCUA"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"mrnarna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">0.069</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 81.57763</code></pre>
<div class="sourceCode" id="cb253"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Na correction -  tanna07 (mRNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"UACGCGUCAAUAACGCUA"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"mrnarna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">0.069</span>, correction.ion <span class="op">=</span> <span class="st">"tanna07"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 81.57763</code></pre>
<div class="sourceCode" id="cb255"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Na correction - default (DNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CCAGCCAGTCTCTCC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnarna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">0.069</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 62.08869</code></pre>
<div class="sourceCode" id="cb257"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Na correction - wet91 (DNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CCAGCCAGTCTCTCC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnarna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">0.069</span>, correction.ion <span class="op">=</span> <span class="st">"wet91"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 62.08869</code></pre>
</div>
<div id="magnesium-corrections" class="section level3">
<h3 class="hasAnchor">
<a href="#magnesium-corrections" class="anchor"></a>Magnesium corrections</h3>
<p>The available correction methods for magnesium concentration are given in <strong>Table 20</strong>.</p>
<p><strong>Table 20</strong>: Details of the corrections for magnesium concentration</p>
<table class="table">
<colgroup>
<col width="16%">
<col width="20%">
<col width="28%">
<col width="34%">
</colgroup>
<thead><tr class="header">
<th align="left">Correction</th>
<th align="left">Type</th>
<th align="left">Limits/Remarks</th>
<th align="left">Reference</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">
<code>owcmg08</code>*</td>
<td align="left">DNA</td>
<td align="left">Mg&gt;=0.0005; Mg&lt;=0.6.</td>
<td align="left"><span class="citation">Owczarzy et al. (2008)</span></td>
</tr>
<tr class="even">
<td align="left"><code>tanmg06</code></td>
<td align="left">DNA</td>
<td align="left">Mg&gt;=0.0001; Mg&lt;=1; Oligomer length superior to 6 base pairs.</td>
<td align="left"><span class="citation">Tan and Chen (2006)</span></td>
</tr>
<tr class="odd">
<td align="left">
<code>tanmg07</code>*</td>
<td align="left">RNA or 2’-O-MeRNA/RNA</td>
<td align="left">Mg&gt;=0.1; Mg&lt;=0.3.</td>
<td align="left"><span class="citation">Tan and Chen (2007)</span></td>
</tr>
</tbody>
</table>
* Default method for computation.

<div class="sourceCode" id="cb259"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Mg correction - default (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAGCCTCGTCGCAGC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Mg.conc <span class="op">=</span> <span class="fl">0.0015</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 65.52043</code></pre>
<div class="sourceCode" id="cb261"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Mg correction - owcmg08 (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAGCCTCGTCGCAGC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Mg.conc <span class="op">=</span> <span class="fl">0.0015</span>, correction.ion <span class="op">=</span> <span class="st">"owcmg08"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 65.52043</code></pre>
<div class="sourceCode" id="cb263"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Mg correction - tanmg06 (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAGCCTCGTCGCAGC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Mg.conc <span class="op">=</span> <span class="fl">0.0015</span>, correction.ion <span class="op">=</span> <span class="st">"tanmg06"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 64.88082</code></pre>
<div class="sourceCode" id="cb265"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Mg correction - default (RNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAGCCUCGUCGCAGC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>,
        Mg.conc <span class="op">=</span> <span class="fl">0.0015</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 82.0796</code></pre>
<div class="sourceCode" id="cb267"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Mg correction -  tanmg07 (RNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAGCCUCGUCGCAGC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>,
        Mg.conc <span class="op">=</span> <span class="fl">0.0015</span>, correction.ion <span class="op">=</span> <span class="st">"tanmg07"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 82.0796</code></pre>
<div class="sourceCode" id="cb269"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Mg correction - default (mRNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"UACGCGUCAAUAACGCUA"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"mrnarna"</span>,
        Mg.conc <span class="op">=</span> <span class="fl">0.0015</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 90.06842</code></pre>
<div class="sourceCode" id="cb271"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Mg correction -  tanmg07 (mRNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"UACGCGUCAAUAACGCUA"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"mrnarna"</span>,
        Mg.conc <span class="op">=</span> <span class="fl">0.0015</span>, correction.ion <span class="op">=</span> <span class="st">"tanmg07"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 90.06842</code></pre>
</div>
<div id="mixed-sodium-and-magnesium-corrections" class="section level3">
<h3 class="hasAnchor">
<a href="#mixed-sodium-and-magnesium-corrections" class="anchor"></a>Mixed Sodium and Magnesium corrections</h3>
<p>The available correction methods for mixed sodium magnesium concentration are given in <strong>Table 21</strong>.</p>
<p><strong>Table 21</strong>: Details of the corrections for mixed sodium and magnesium concentration</p>
<table class="table">
<colgroup>
<col width="17%">
<col width="20%">
<col width="28%">
<col width="34%">
</colgroup>
<thead><tr class="header">
<th align="left">Correction</th>
<th align="left">Type</th>
<th align="left">Limits/Remarks</th>
<th align="left">Reference</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">
<code>owcmix08</code>*</td>
<td align="left">DNA</td>
<td align="left">Mg&gt;=0.0005; Mg&lt;=0.6; Na+K+Tris/2&gt;0.</td>
<td align="left"><span class="citation">Owczarzy et al. (2008)</span></td>
</tr>
<tr class="even">
<td align="left"><code>tanmix07</code></td>
<td align="left">DNA, RNA or 2’-O-MeRNA/RNA</td>
<td align="left">Mg&gt;=0.1; Mg&lt;=0.3; Na+K+Tris/2&gt;=0.1; Na+K+Tris/2&lt;=0.3.</td>
<td align="left"><span class="citation">Tan and Chen (2007)</span></td>
</tr>
</tbody>
</table>
* Default method for computation.

<div class="sourceCode" id="cb273"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Mixed Na &amp; Mg correction - default (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAGCCTCGTCGCAGC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">0.069</span>, Mg.conc <span class="op">=</span> <span class="fl">0.0015</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 65.83371</code></pre>
<div class="sourceCode" id="cb275"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Mixed Na &amp; Mg correction - owcmix08 (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAGCCTCGTCGCAGC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">0.069</span>, Mg.conc <span class="op">=</span> <span class="fl">0.0015</span>, correction.ion <span class="op">=</span> <span class="st">"owcmix08"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 65.83371</code></pre>
<div class="sourceCode" id="cb277"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Mixed Na &amp; Mg correction - tanmix07 (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAGCCTCGTCGCAGC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">0.069</span>, Mg.conc <span class="op">=</span> <span class="fl">0.0015</span>, correction.ion <span class="op">=</span> <span class="st">"tanmix07"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 63.21723</code></pre>
<div class="sourceCode" id="cb279"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Mixed Na &amp; Mg correction - default (RNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAGCCUCGUCGCAGC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">0.069</span>, Mg.conc <span class="op">=</span> <span class="fl">0.0015</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 79.40119</code></pre>
<div class="sourceCode" id="cb281"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Mixed Na &amp; Mg correction - tanmix07 (RNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAGCCUCGUCGCAGC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">0.069</span>, Mg.conc <span class="op">=</span> <span class="fl">0.0015</span>, correction.ion <span class="op">=</span> <span class="st">"tanmix07"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 79.40119</code></pre>
<div class="sourceCode" id="cb283"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Mixed Na &amp; Mg correction - default (mRNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"UACGCGUCAAUAACGCUA"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"mrnarna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">0.069</span>, Mg.conc <span class="op">=</span> <span class="fl">0.0015</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 96.46186</code></pre>
<div class="sourceCode" id="cb285"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Mixed Na &amp; Mg correction - tanmix07 (mRNA/RNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"UACGCGUCAAUAACGCUA"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"mrnarna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">0.069</span>, Mg.conc <span class="op">=</span> <span class="fl">0.0015</span>, correction.ion <span class="op">=</span> <span class="st">"tanmix07"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 96.46186</code></pre>
<p>The ion correction by <span class="citation">Owczarzy et al. (2008)</span> is used by default according to the <span class="math inline">\(\frac{[\textrm{Mg}^{2+}]^{0.5}}{[\textrm{Mon}^{+}]}\)</span> ratio, where <span class="math inline">\([\textrm{Mon}^{+}] = \textrm{Na}^{+}]+[\textrm{Tris}^{+}]+[\textrm{K}^{+}]\)</span>.</p>
<p>If,</p>
<ul>
<li>
<span class="math inline">\([\textrm{K}^{+}] = 0\)</span>, default sodium correction is used;</li>
<li>Ratio <span class="math inline">\(&lt;\)</span> 0.22, default sodium correction is used;</li>
<li>0.22 <span class="math inline">\(\leq\)</span> Ratio <span class="math inline">\(&lt;\)</span> 6, default mixed Na and Mg correction is used and</li>
<li>Ratio <span class="math inline">\(\geq\)</span> 6,default magnesium correction is used.</li>
</ul>
<p>Note that <span class="math inline">\([\textrm{Tris}^{+}]\)</span> is about half of the total tris buffer concentration.</p>
</div>
<div id="sodium-equivalent-concentration-methods" class="section level3">
<h3 class="hasAnchor">
<a href="#sodium-equivalent-concentration-methods" class="anchor"></a>Sodium equivalent concentration methods</h3>
<p>The available correction methods for mixed sodium magnesium concentration are given in <strong>Table 22</strong>.</p>
<p><strong>Table 22</strong>: Details of the methods for computation of sodium equivalent concentration in the presence of other ions</p>
<table class="table">
<colgroup>
<col width="18%">
<col width="9%">
<col width="23%">
<col width="47%">
</colgroup>
<thead><tr class="header">
<th align="left">Correction</th>
<th align="left">Type</th>
<th align="left">Limits/Remarks</th>
<th align="left">Reference</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">
<code>ahs01</code>*</td>
<td align="left">DNA</td>
<td align="left"></td>
<td align="left"><span class="citation">Ahsen et al. (2001)</span></td>
</tr>
<tr class="even">
<td align="left"><code>mit96</code></td>
<td align="left">DNA</td>
<td align="left"></td>
<td align="left"><span class="citation">Mitsuhashi (1996)</span></td>
</tr>
<tr class="odd">
<td align="left"><code>pey00</code></td>
<td align="left">DNA</td>
<td align="left"></td>
<td align="left"><span class="citation">Peyret (2000)</span></td>
</tr>
</tbody>
</table>
* Default method for computation.

<div class="sourceCode" id="cb287"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Na equivalent concentration method - default (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAGCCTCGTCGCAGC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">0.069</span>, Mg.conc <span class="op">=</span> <span class="fl">0.0015</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 65.83371</code></pre>
<div class="sourceCode" id="cb289"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Na equivalent concentration method - ahs01 (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAGCCTCGTCGCAGC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">0.069</span>, Mg.conc <span class="op">=</span> <span class="fl">0.0015</span>, method.Naeq <span class="op">=</span> <span class="st">"ahs01"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 65.83371</code></pre>
<div class="sourceCode" id="cb291"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Na equivalent concentration method - mit96 (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAGCCTCGTCGCAGC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">0.069</span>, Mg.conc <span class="op">=</span> <span class="fl">0.0015</span>, method.Naeq <span class="op">=</span> <span class="st">"mit96"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 65.83371</code></pre>
<div class="sourceCode" id="cb293"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Na equivalent concentration method - pey00 (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAGCCTCGTCGCAGC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">0.069</span>, Mg.conc <span class="op">=</span> <span class="fl">0.0015</span>, method.Naeq <span class="op">=</span> <span class="st">"pey00"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 65.83371</code></pre>
</div>
</div>
<div id="denaturing-agent-corrections" class="section level2">
<h2 class="hasAnchor">
<a href="#denaturing-agent-corrections" class="anchor"></a>Denaturing agent corrections</h2>
<p>These include melting temperature corrections for concentration of formamide and DMSO.</p>
<div id="dmso-corrections" class="section level3">
<h3 class="hasAnchor">
<a href="#dmso-corrections" class="anchor"></a>DMSO corrections</h3>
<p>The available correction methods for DMSO concentration are given in <strong>Table 23</strong>.</p>
<p><strong>Table 23</strong>: Details of the corrections for DMSO concentration</p>
<table class="table">
<colgroup>
<col width="18%">
<col width="9%">
<col width="25%">
<col width="47%">
</colgroup>
<thead><tr class="header">
<th align="left">Correction</th>
<th align="left">Type</th>
<th align="left">Limits/Remarks</th>
<th align="left">Reference</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">
<code>ahs01</code>*</td>
<td align="left">DNA</td>
<td align="left">Not tested with experimental results.</td>
<td align="left"><span class="citation">Ahsen et al. (2001)</span></td>
</tr>
<tr class="even">
<td align="left"><code>cul76</code></td>
<td align="left">DNA</td>
<td align="left">Not tested with experimental results.</td>
<td align="left"><span class="citation">Cullen and Bick (1976)</span></td>
</tr>
<tr class="odd">
<td align="left"><code>esc80</code></td>
<td align="left">DNA</td>
<td align="left">Not tested with experimental results.</td>
<td align="left"><span class="citation">Escara and Hutton (1980)</span></td>
</tr>
<tr class="even">
<td align="left"><code>mus81</code></td>
<td align="left">DNA</td>
<td align="left">Not tested with experimental results.</td>
<td align="left"><span class="citation">Musielski et al. (1981)</span></td>
</tr>
</tbody>
</table>
* Default method for computation.

<div class="sourceCode" id="cb295"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># DMSO correction - default (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAGCCTCGTCGCAGC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, DMSO.conc <span class="op">=</span> <span class="fl">10</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 65.40154</code></pre>
<div class="sourceCode" id="cb297"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># DMSO correction - ahs01 (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAGCCTCGTCGCAGC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, DMSO.conc <span class="op">=</span> <span class="fl">10</span>, correction.DMSO <span class="op">=</span> <span class="st">"ahs01"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 65.40154</code></pre>
<div class="sourceCode" id="cb299"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># DMSO correction - cul76 (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAGCCTCGTCGCAGC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, DMSO.conc <span class="op">=</span> <span class="fl">10</span>, correction.DMSO <span class="op">=</span> <span class="st">"cul76"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 67.90154</code></pre>
<div class="sourceCode" id="cb301"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># DMSO correction - esc80 (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAGCCTCGTCGCAGC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, DMSO.conc <span class="op">=</span> <span class="fl">10</span>, correction.DMSO <span class="op">=</span> <span class="st">"esc80"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 66.15154</code></pre>
<div class="sourceCode" id="cb303"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># DMSO correction - mus80 (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAGCCTCGTCGCAGC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, DMSO.conc <span class="op">=</span> <span class="fl">10</span>, correction.DMSO <span class="op">=</span> <span class="st">"mus81"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 66.90154</code></pre>
</div>
<div id="formamide-corrections" class="section level3">
<h3 class="hasAnchor">
<a href="#formamide-corrections" class="anchor"></a>Formamide corrections</h3>
<p>The available correction methods for formamide concentration are given in <strong>Table 24</strong>.</p>
<p><strong>Table 24</strong>: Details of the corrections for formamide concentration</p>
<table class="table">
<colgroup>
<col width="18%">
<col width="9%">
<col width="27%">
<col width="41%">
</colgroup>
<thead><tr class="header">
<th align="left">Correction</th>
<th align="left">Type</th>
<th align="left">Limits/Remarks</th>
<th align="left">Reference</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">
<code>bla96</code>*</td>
<td align="left">DNA</td>
<td align="left">With formamide concentration in mol/L.</td>
<td align="left"><span class="citation">Blake (1996)</span></td>
</tr>
<tr class="even">
<td align="left"><code>lincorr</code></td>
<td align="left">DNA</td>
<td align="left">With a % of formamide volume.</td>
<td align="left">
<span class="citation">McConaughy et al. (1969)</span>, <span class="citation">Record (1967)</span>, <span class="citation">Casey and Davidson (1977)</span>, <span class="citation">Hutton (1977)</span>
</td>
</tr>
</tbody>
</table>
* Default method for computation.

<div class="sourceCode" id="cb305"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Formamide correction - default (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAGCCTCGTCGCAGC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, formamide.conc <span class="op">=</span> <span class="fl">0.06</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 72.74867</code></pre>
<div class="sourceCode" id="cb307"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Formamide correction - bla96 (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAGCCTCGTCGCAGC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, formamide.conc <span class="op">=</span> <span class="fl">0.06</span>, correction.formamide <span class="op">=</span> <span class="st">"bla96"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 72.74867</code></pre>
<div class="sourceCode" id="cb309"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Formamide correction - lincorr (DNA/DNA)</span>
<span class="fu"><a href="../reference/melting.html">melting</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="st">"CAGCCTCGTCGCAGC"</span>,
        nucleic.acid.conc <span class="op">=</span> <span class="fl">0.000002</span>, hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>,
        Na.conc <span class="op">=</span> <span class="fl">1</span>, formamide.conc <span class="op">=</span> <span class="fl">10</span>, correction.formamide <span class="op">=</span> <span class="st">"lincorr"</span><span class="op">)</span></code></pre></div>
<pre><code>## [1] 66.40154</code></pre>
</div>
</div>
</div>
<div id="equivalent-options-in-melting-5" class="section level1">
<h1 class="hasAnchor">
<a href="#equivalent-options-in-melting-5" class="anchor"></a>Equivalent options in MELTING 5</h1>
<p>The options in MELTING 5 command line equivalent to the arguments in <code>rmelting</code> are given in <strong>Table 25</strong>.</p>
<p><strong>Table 24</strong>: Arguments in <code>rmelting</code> and their equivalent options in MELTING 5 command line.</p>
<pre><code>## New names:
## * `` -&gt; ...3
## * `` -&gt; ...4</code></pre>
<table class="table">
<thead><tr class="header">
<th align="left"><code>rmelting</code></th>
<th align="left">MELTING 5 (command line)</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left"><code>sequence</code></td>
<td align="left">-S</td>
</tr>
<tr class="even">
<td align="left"><code>comp.sequence</code></td>
<td align="left">-C</td>
</tr>
<tr class="odd">
<td align="left"><code>nucleic.acid.conc</code></td>
<td align="left">-P</td>
</tr>
<tr class="even">
<td align="left"><code>hybridisation.type</code></td>
<td align="left">-H</td>
</tr>
<tr class="odd">
<td align="left"><code>Na.conc</code></td>
<td align="left">-E</td>
</tr>
<tr class="even">
<td align="left"><code>Mg.conc</code></td>
<td align="left">-E</td>
</tr>
<tr class="odd">
<td align="left"><code>Tris.conc</code></td>
<td align="left">-E</td>
</tr>
<tr class="even">
<td align="left"><code>K.conc</code></td>
<td align="left">-E</td>
</tr>
<tr class="odd">
<td align="left"><code>dNTP.conc</code></td>
<td align="left">-E</td>
</tr>
<tr class="even">
<td align="left"><code>DMSO.conc</code></td>
<td align="left">-E</td>
</tr>
<tr class="odd">
<td align="left"><code>formamide.conc</code></td>
<td align="left">-E</td>
</tr>
<tr class="even">
<td align="left"><code>size.threshold</code></td>
<td align="left">-T</td>
</tr>
<tr class="odd">
<td align="left"><code>self</code></td>
<td align="left">-self</td>
</tr>
<tr class="even">
<td align="left"><code>correction.factor</code></td>
<td align="left">-F</td>
</tr>
<tr class="odd">
<td align="left"><code>method.approx</code></td>
<td align="left">-am</td>
</tr>
<tr class="even">
<td align="left"><code>method.nn</code></td>
<td align="left">-nn</td>
</tr>
<tr class="odd">
<td align="left"><code>method.GU</code></td>
<td align="left">-GU</td>
</tr>
<tr class="even">
<td align="left"><code>method.singleMM</code></td>
<td align="left">-sinMM</td>
</tr>
<tr class="odd">
<td align="left"><code>method.tandemMM</code></td>
<td align="left">-tanMM</td>
</tr>
<tr class="even">
<td align="left"><code>method.single.dangle</code></td>
<td align="left">-sinDE</td>
</tr>
<tr class="odd">
<td align="left"><code>method.double.dangle</code></td>
<td align="left">-secDE</td>
</tr>
<tr class="even">
<td align="left"><code>method.long.dangle</code></td>
<td align="left">-lonDE</td>
</tr>
<tr class="odd">
<td align="left"><code>method.internal.loop</code></td>
<td align="left">-intLP</td>
</tr>
<tr class="even">
<td align="left"><code>method.single.bulge.loop</code></td>
<td align="left">-sinBU</td>
</tr>
<tr class="odd">
<td align="left"><code>method.long.bulge.loop</code></td>
<td align="left">-lonBU</td>
</tr>
<tr class="even">
<td align="left"><code>method.CNG</code></td>
<td align="left">-CNG</td>
</tr>
<tr class="odd">
<td align="left"><code>method.inosine</code></td>
<td align="left">-ino</td>
</tr>
<tr class="even">
<td align="left"><code>method.hydroxyadenine</code></td>
<td align="left">-ha</td>
</tr>
<tr class="odd">
<td align="left"><code>method.azobenzenes</code></td>
<td align="left">-azo</td>
</tr>
<tr class="even">
<td align="left"><code>method.locked</code></td>
<td align="left">-lck</td>
</tr>
<tr class="odd">
<td align="left"><code>correction.ion</code></td>
<td align="left">-ion</td>
</tr>
<tr class="even">
<td align="left"><code>method.Naeq</code></td>
<td align="left">-naeq</td>
</tr>
<tr class="odd">
<td align="left"><code>correction.DMSO</code></td>
<td align="left">-DMSO</td>
</tr>
<tr class="even">
<td align="left"><code>correction.formamide</code></td>
<td align="left">-for</td>
</tr>
</tbody>
</table>
</div>
<div id="batch-computation" class="section level1">
<h1 class="hasAnchor">
<a href="#batch-computation" class="anchor"></a>Batch computation</h1>
<p>Melting temperature for multiple nucleic acid duplexes can be computed using the <code>meltingBatch</code> function.</p>
<div class="sourceCode" id="cb312"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">sequence</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span><span class="st">"CAAAAAG"</span>, <span class="st">"CAAAAAAG"</span>, <span class="st">"TTTTATAATAAA"</span>, <span class="st">"CCATCGCTACC"</span>,
              <span class="st">"CAAACAAAG"</span>, <span class="st">"CCATTGCTACC"</span>, <span class="st">"CAAAAAAAG"</span>, <span class="st">"GTGAAC"</span>, <span class="st">"AAAAAAAA"</span>,
              <span class="st">"CAACTTGATATTATTA"</span>, <span class="st">"CAAATAAAG"</span>, <span class="st">"GCGAGC"</span>, <span class="st">"GGGACC"</span>,
              <span class="st">"CAAAGAAAG"</span>, <span class="st">"CTGACAAGTGTC"</span>, <span class="st">"GCGAAAAGCG"</span><span class="op">)</span>

<span class="fu"><a href="../reference/meltingBatch.html">meltingBatch</a></span><span class="op">(</span><span class="va">sequence</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0004</span>,
             hybridisation.type <span class="op">=</span> <span class="st">"dnadna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span><span class="op">)</span></code></pre></div>
<pre><code>##       Environment.Sequence Environment.Complementary sequence
##  [1,] "CAAAAAG"            "GTTTTTC"                         
##  [2,] "CAAAAAAG"           "GTTTTTTC"                        
##  [3,] "TTTTATAATAAA"       "AAAATATTATTT"                    
##  [4,] "CCATCGCTACC"        "GGTAGCGATGG"                     
##  [5,] "CAAACAAAG"          "GTTTGTTTC"                       
##  [6,] "CCATTGCTACC"        "GGTAACGATGG"                     
##  [7,] "CAAAAAAAG"          "GTTTTTTTC"                       
##  [8,] "GTGAAC"             "CACTTG"                          
##  [9,] "AAAAAAAA"           "TTTTTTTT"                        
## [10,] "CAACTTGATATTATTA"   "GTTGAACTATAATAAT"                
## [11,] "CAAATAAAG"          "GTTTATTTC"                       
## [12,] "GCGAGC"             "CGCTCG"                          
## [13,] "GGGACC"             "CCCTGG"                          
## [14,] "CAAAGAAAG"          "GTTTCTTTC"                       
## [15,] "CTGACAAGTGTC"       "GACTGTTCACAG"                    
## [16,] "GCGAAAAGCG"         "CGCTTTTCGC"                      
##       Environment.Nucleic acid concentration (M) Environment.Hybridization type
##  [1,] "4e-04"                                    "dnadna"                      
##  [2,] "4e-04"                                    "dnadna"                      
##  [3,] "4e-04"                                    "dnadna"                      
##  [4,] "4e-04"                                    "dnadna"                      
##  [5,] "4e-04"                                    "dnadna"                      
##  [6,] "4e-04"                                    "dnadna"                      
##  [7,] "4e-04"                                    "dnadna"                      
##  [8,] "4e-04"                                    "dnadna"                      
##  [9,] "4e-04"                                    "dnadna"                      
## [10,] "4e-04"                                    "dnadna"                      
## [11,] "4e-04"                                    "dnadna"                      
## [12,] "4e-04"                                    "dnadna"                      
## [13,] "4e-04"                                    "dnadna"                      
## [14,] "4e-04"                                    "dnadna"                      
## [15,] "4e-04"                                    "dnadna"                      
## [16,] "4e-04"                                    "dnadna"                      
##       Environment.Na concentration (M) Environment.Mg concentration (M)
##  [1,] "1"                              "0"                             
##  [2,] "1"                              "0"                             
##  [3,] "1"                              "0"                             
##  [4,] "1"                              "0"                             
##  [5,] "1"                              "0"                             
##  [6,] "1"                              "0"                             
##  [7,] "1"                              "0"                             
##  [8,] "1"                              "0"                             
##  [9,] "1"                              "0"                             
## [10,] "1"                              "0"                             
## [11,] "1"                              "0"                             
## [12,] "1"                              "0"                             
## [13,] "1"                              "0"                             
## [14,] "1"                              "0"                             
## [15,] "1"                              "0"                             
## [16,] "1"                              "0"                             
##       Environment.Tris concentration (M) Environment.K concentration (M)
##  [1,] "0"                                "0"                            
##  [2,] "0"                                "0"                            
##  [3,] "0"                                "0"                            
##  [4,] "0"                                "0"                            
##  [5,] "0"                                "0"                            
##  [6,] "0"                                "0"                            
##  [7,] "0"                                "0"                            
##  [8,] "0"                                "0"                            
##  [9,] "0"                                "0"                            
## [10,] "0"                                "0"                            
## [11,] "0"                                "0"                            
## [12,] "0"                                "0"                            
## [13,] "0"                                "0"                            
## [14,] "0"                                "0"                            
## [15,] "0"                                "0"                            
## [16,] "0"                                "0"                            
##       Environment.dNTP concentration (M) Environment.DMSO concentration (%)
##  [1,] "0"                                "0"                               
##  [2,] "0"                                "0"                               
##  [3,] "0"                                "0"                               
##  [4,] "0"                                "0"                               
##  [5,] "0"                                "0"                               
##  [6,] "0"                                "0"                               
##  [7,] "0"                                "0"                               
##  [8,] "0"                                "0"                               
##  [9,] "0"                                "0"                               
## [10,] "0"                                "0"                               
## [11,] "0"                                "0"                               
## [12,] "0"                                "0"                               
## [13,] "0"                                "0"                               
## [14,] "0"                                "0"                               
## [15,] "0"                                "0"                               
## [16,] "0"                                "0"                               
##       Environment.Formamide concentration (M or %)
##  [1,] "0"                                         
##  [2,] "0"                                         
##  [3,] "0"                                         
##  [4,] "0"                                         
##  [5,] "0"                                         
##  [6,] "0"                                         
##  [7,] "0"                                         
##  [8,] "0"                                         
##  [9,] "0"                                         
## [10,] "0"                                         
## [11,] "0"                                         
## [12,] "0"                                         
## [13,] "0"                                         
## [14,] "0"                                         
## [15,] "0"                                         
## [16,] "0"                                         
##       Environment.Self complementarity Environment.Correction factor
##  [1,] "FALSE"                          "4"                          
##  [2,] "FALSE"                          "4"                          
##  [3,] "FALSE"                          "4"                          
##  [4,] "FALSE"                          "4"                          
##  [5,] "FALSE"                          "4"                          
##  [6,] "FALSE"                          "4"                          
##  [7,] "FALSE"                          "4"                          
##  [8,] "FALSE"                          "4"                          
##  [9,] "FALSE"                          "4"                          
## [10,] "FALSE"                          "4"                          
## [11,] "FALSE"                          "4"                          
## [12,] "FALSE"                          "4"                          
## [13,] "FALSE"                          "4"                          
## [14,] "FALSE"                          "4"                          
## [15,] "FALSE"                          "4"                          
## [16,] "FALSE"                          "4"                          
##       Options.Approximative formula Options.Nearest neighbour model
##  [1,] NA                            "all97"                        
##  [2,] NA                            "all97"                        
##  [3,] NA                            "all97"                        
##  [4,] NA                            "all97"                        
##  [5,] NA                            "all97"                        
##  [6,] NA                            "all97"                        
##  [7,] NA                            "all97"                        
##  [8,] NA                            "all97"                        
##  [9,] NA                            "all97"                        
## [10,] NA                            "all97"                        
## [11,] NA                            "all97"                        
## [12,] NA                            "all97"                        
## [13,] NA                            "all97"                        
## [14,] NA                            "all97"                        
## [15,] NA                            "all97"                        
## [16,] NA                            "all97"                        
##       Options.GU model Options.Single mismatch model
##  [1,] NA               "allsanpey"                  
##  [2,] NA               "allsanpey"                  
##  [3,] NA               "allsanpey"                  
##  [4,] NA               "allsanpey"                  
##  [5,] NA               "allsanpey"                  
##  [6,] NA               "allsanpey"                  
##  [7,] NA               "allsanpey"                  
##  [8,] NA               "allsanpey"                  
##  [9,] NA               "allsanpey"                  
## [10,] NA               "allsanpey"                  
## [11,] NA               "allsanpey"                  
## [12,] NA               "allsanpey"                  
## [13,] NA               "allsanpey"                  
## [14,] NA               "allsanpey"                  
## [15,] NA               "allsanpey"                  
## [16,] NA               "allsanpey"                  
##       Options.Tandem mismatch model Options.Single dangling end model
##  [1,] "allsanpey"                   "bom00"                          
##  [2,] "allsanpey"                   "bom00"                          
##  [3,] "allsanpey"                   "bom00"                          
##  [4,] "allsanpey"                   "bom00"                          
##  [5,] "allsanpey"                   "bom00"                          
##  [6,] "allsanpey"                   "bom00"                          
##  [7,] "allsanpey"                   "bom00"                          
##  [8,] "allsanpey"                   "bom00"                          
##  [9,] "allsanpey"                   "bom00"                          
## [10,] "allsanpey"                   "bom00"                          
## [11,] "allsanpey"                   "bom00"                          
## [12,] "allsanpey"                   "bom00"                          
## [13,] "allsanpey"                   "bom00"                          
## [14,] "allsanpey"                   "bom00"                          
## [15,] "allsanpey"                   "bom00"                          
## [16,] "allsanpey"                   "bom00"                          
##       Options.Double dangling end model Options.Long dangling end model
##  [1,] "sugdna02"                        "sugdna02"                     
##  [2,] "sugdna02"                        "sugdna02"                     
##  [3,] "sugdna02"                        "sugdna02"                     
##  [4,] "sugdna02"                        "sugdna02"                     
##  [5,] "sugdna02"                        "sugdna02"                     
##  [6,] "sugdna02"                        "sugdna02"                     
##  [7,] "sugdna02"                        "sugdna02"                     
##  [8,] "sugdna02"                        "sugdna02"                     
##  [9,] "sugdna02"                        "sugdna02"                     
## [10,] "sugdna02"                        "sugdna02"                     
## [11,] "sugdna02"                        "sugdna02"                     
## [12,] "sugdna02"                        "sugdna02"                     
## [13,] "sugdna02"                        "sugdna02"                     
## [14,] "sugdna02"                        "sugdna02"                     
## [15,] "sugdna02"                        "sugdna02"                     
## [16,] "sugdna02"                        "sugdna02"                     
##       Options.Internal loop model Options.Single bulge loop model
##  [1,] "san04"                     "tan04"                        
##  [2,] "san04"                     "tan04"                        
##  [3,] "san04"                     "tan04"                        
##  [4,] "san04"                     "tan04"                        
##  [5,] "san04"                     "tan04"                        
##  [6,] "san04"                     "tan04"                        
##  [7,] "san04"                     "tan04"                        
##  [8,] "san04"                     "tan04"                        
##  [9,] "san04"                     "tan04"                        
## [10,] "san04"                     "tan04"                        
## [11,] "san04"                     "tan04"                        
## [12,] "san04"                     "tan04"                        
## [13,] "san04"                     "tan04"                        
## [14,] "san04"                     "tan04"                        
## [15,] "san04"                     "tan04"                        
## [16,] "san04"                     "tan04"                        
##       Options.Long bulge loop model Options.CNG repeats model
##  [1,] "san04"                       NA                       
##  [2,] "san04"                       NA                       
##  [3,] "san04"                       NA                       
##  [4,] "san04"                       NA                       
##  [5,] "san04"                       NA                       
##  [6,] "san04"                       NA                       
##  [7,] "san04"                       NA                       
##  [8,] "san04"                       NA                       
##  [9,] "san04"                       NA                       
## [10,] "san04"                       NA                       
## [11,] "san04"                       NA                       
## [12,] "san04"                       NA                       
## [13,] "san04"                       NA                       
## [14,] "san04"                       NA                       
## [15,] "san04"                       NA                       
## [16,] "san04"                       NA                       
##       Options.Inosine bases model Options.Hydroxyadenine bases model
##  [1,] "san05"                     "sug01"                           
##  [2,] "san05"                     "sug01"                           
##  [3,] "san05"                     "sug01"                           
##  [4,] "san05"                     "sug01"                           
##  [5,] "san05"                     "sug01"                           
##  [6,] "san05"                     "sug01"                           
##  [7,] "san05"                     "sug01"                           
##  [8,] "san05"                     "sug01"                           
##  [9,] "san05"                     "sug01"                           
## [10,] "san05"                     "sug01"                           
## [11,] "san05"                     "sug01"                           
## [12,] "san05"                     "sug01"                           
## [13,] "san05"                     "sug01"                           
## [14,] "san05"                     "sug01"                           
## [15,] "san05"                     "sug01"                           
## [16,] "san05"                     "sug01"                           
##       Options.Azobenzenes model Options.Locked nucleic acids model
##  [1,] "asa05"                   "mct04"                           
##  [2,] "asa05"                   "mct04"                           
##  [3,] "asa05"                   "mct04"                           
##  [4,] "asa05"                   "mct04"                           
##  [5,] "asa05"                   "mct04"                           
##  [6,] "asa05"                   "mct04"                           
##  [7,] "asa05"                   "mct04"                           
##  [8,] "asa05"                   "mct04"                           
##  [9,] "asa05"                   "mct04"                           
## [10,] "asa05"                   "mct04"                           
## [11,] "asa05"                   "mct04"                           
## [12,] "asa05"                   "mct04"                           
## [13,] "asa05"                   "mct04"                           
## [14,] "asa05"                   "mct04"                           
## [15,] "asa05"                   "mct04"                           
## [16,] "asa05"                   "mct04"                           
##       Options.Ion correction method Options.Na equivalence correction method
##  [1,] NA                            "ahs01"                                 
##  [2,] NA                            "ahs01"                                 
##  [3,] NA                            "ahs01"                                 
##  [4,] NA                            "ahs01"                                 
##  [5,] NA                            "ahs01"                                 
##  [6,] NA                            "ahs01"                                 
##  [7,] NA                            "ahs01"                                 
##  [8,] NA                            "ahs01"                                 
##  [9,] NA                            "ahs01"                                 
## [10,] NA                            "ahs01"                                 
## [11,] NA                            "ahs01"                                 
## [12,] NA                            "ahs01"                                 
## [13,] NA                            "ahs01"                                 
## [14,] NA                            "ahs01"                                 
## [15,] NA                            "ahs01"                                 
## [16,] NA                            "ahs01"                                 
##       Options.DMSO correction method Options.Formamide correction method
##  [1,] "ahs01"                        "bla96"                            
##  [2,] "ahs01"                        "bla96"                            
##  [3,] "ahs01"                        "bla96"                            
##  [4,] "ahs01"                        "bla96"                            
##  [5,] "ahs01"                        "bla96"                            
##  [6,] "ahs01"                        "bla96"                            
##  [7,] "ahs01"                        "bla96"                            
##  [8,] "ahs01"                        "bla96"                            
##  [9,] "ahs01"                        "bla96"                            
## [10,] "ahs01"                        "bla96"                            
## [11,] "ahs01"                        "bla96"                            
## [12,] "ahs01"                        "bla96"                            
## [13,] "ahs01"                        "bla96"                            
## [14,] "ahs01"                        "bla96"                            
## [15,] "ahs01"                        "bla96"                            
## [16,] "ahs01"                        "bla96"                            
##       Options.Mode Results.Enthalpy (cal) Results.Entropy (cal)
##  [1,] "def"        "-47700"               "-138.1"             
##  [2,] "def"        "-55600"               "-160.3"             
##  [3,] "def"        "-78800"               "-229.7"             
##  [4,] "def"        "-83500"               "-227"               
##  [5,] "def"        "-64600"               "-183.2"             
##  [6,] "def"        "-81100"               "-222.5"             
##  [7,] "def"        "-63500"               "-182.5"             
##  [8,] "def"        "-41200"               "-117.5"             
##  [9,] "def"        "-50700"               "-147.2"             
## [10,] "def"        "-113800"              "-323.6"             
## [11,] "def"        "-62100"               "-179.8"             
## [12,] "def"        "-46000"               "-124.8"             
## [13,] "def"        "-40400"               "-109.9"             
## [14,] "def"        "-63700"               "-181.3"             
## [15,] "def"        "-90400"               "-249.5"             
## [16,] "def"        "-80300"               "-218.6"             
##       Results.Enthalpy (J) Results.Entropy (J) Results.Melting temperature (C)
##  [1,] "-199386"            "-577.258"          "31.7814953255144"             
##  [2,] "-232408"            "-670.054"          "38.1103863719918"             
##  [3,] "-329384"            "-960.146"          "44.5553259145469"             
##  [4,] "-349030"            "-948.86"           "67.2098590318261"             
##  [5,] "-270028"            "-765.776"          "47.400072116762"              
##  [6,] "-338998"            "-930.05"           "63.6040550863501"             
##  [7,] "-265430"            "-762.85"           "43.0400604037136"             
##  [8,] "-172216"            "-491.15"           "30.1735038367475"             
##  [9,] "-211926"            "-615.296"          "33.1415226764116"             
## [10,] "-475684"            "-1352.648"         "59.6680431282422"             
## [11,] "-259578"            "-751.564"          "40.2828264688437"             
## [12,] "-192280"            "-521.664"          "48.2393469411973"             
## [13,] "-168872"            "-459.382"          "41.9123740666287"             
## [14,] "-266266"            "-757.834"          "45.9425910944819"             
## [15,] "-377872"            "-1042.91"          "64.379329012421"              
## [16,] "-335654"            "-913.748"          "65.7707030297917"             
##       Message
##  [1,] NA     
##  [2,] NA     
##  [3,] NA     
##  [4,] NA     
##  [5,] NA     
##  [6,] NA     
##  [7,] NA     
##  [8,] NA     
##  [9,] NA     
## [10,] NA     
## [11,] NA     
## [12,] NA     
## [13,] NA     
## [14,] NA     
## [15,] NA     
## [16,] NA</code></pre>
<p>Complementary sequences are computed by default, but need to be specified in case of mismatches, inosine(s) or hydroxyadenine(s) between the two strands.</p>
<div class="sourceCode" id="cb314"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">seq</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span><span class="st">"GCAUACG"</span>, <span class="st">"CAGUAGGUC"</span>, <span class="st">"CGCUCGC"</span>, <span class="st">"GAGUGGAG"</span>, <span class="st">"GACAGGCUG"</span>,
         <span class="st">"CAGUACGUC"</span>, <span class="st">"GACAUCCUG"</span>, <span class="st">"GACCACCUG"</span>, <span class="st">"CAGAAUGUC"</span>, <span class="st">"GCGUCGC"</span>,
         <span class="st">"CGUCCGG"</span>, <span class="st">"GACUCUCUG"</span>, <span class="st">"CAGCUGGUC"</span>, <span class="st">"GACUAGCUG"</span>, <span class="st">"CUCUGCUC"</span>,
         <span class="st">"GCGUCCG"</span>, <span class="st">"GUCCGCG"</span>, <span class="st">"CGAUCAC"</span>, <span class="st">"GACUACCUG"</span>, <span class="st">"GACGAUCUG"</span><span class="op">)</span>

<span class="va">comp.seq</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span><span class="st">"CGUUUGC"</span>, <span class="st">"GUCGGCCAG"</span>, <span class="st">"GCGUGCG"</span>, <span class="st">"CUCUUCUC"</span>, <span class="st">"CUGUGCGAC"</span>,
              <span class="st">"GUCGGGCAG"</span>, <span class="st">"CUGUUGGAC"</span>, <span class="st">"CUGGGGGAC"</span>, <span class="st">"GUCUGGCAG"</span>, <span class="st">"CGCUGCG"</span>,
              <span class="st">"GCUGGCC"</span>, <span class="st">"CUGAUAGAC"</span>, <span class="st">"GUCGUUCAG"</span>, <span class="st">"CUGAGCGAC"</span>, <span class="st">"GAGUUGAG"</span>,
              <span class="st">"CGCUGGC"</span>, <span class="st">"CUGGCGC"</span>, <span class="st">"GCUUGUG"</span>, <span class="st">"CUGAGGGAC"</span>, <span class="st">"CUGCCAGAC"</span><span class="op">)</span>

<span class="fu"><a href="../reference/meltingBatch.html">meltingBatch</a></span><span class="op">(</span>sequence <span class="op">=</span> <span class="va">seq</span>, comp.seq <span class="op">=</span> <span class="va">comp.seq</span>, nucleic.acid.conc <span class="op">=</span> <span class="fl">0.0004</span>,
             hybridisation.type <span class="op">=</span> <span class="st">"rnarna"</span>, Na.conc <span class="op">=</span> <span class="fl">1</span>,
             method.singleMM <span class="op">=</span> <span class="st">"tur06"</span><span class="op">)</span></code></pre></div>
<pre><code>##       Environment.Sequence Environment.Complementary sequence
##  [1,] "GCAUACG"            "CGUUUGC"                         
##  [2,] "CAGUAGGUC"          "GUCGGCCAG"                       
##  [3,] "CGCUCGC"            "GCGUGCG"                         
##  [4,] "GAGUGGAG"           "CUCUUCUC"                        
##  [5,] "GACAGGCUG"          "CUGUGCGAC"                       
##  [6,] "CAGUACGUC"          "GUCGGGCAG"                       
##  [7,] "GACAUCCUG"          "CUGUUGGAC"                       
##  [8,] "GACCACCUG"          "CUGGGGGAC"                       
##  [9,] "CAGAAUGUC"          "GUCUGGCAG"                       
## [10,] "GCGUCGC"            "CGCUGCG"                         
## [11,] "CGUCCGG"            "GCUGGCC"                         
## [12,] "GACUCUCUG"          "CUGAUAGAC"                       
## [13,] "CAGCUGGUC"          "GUCGUUCAG"                       
## [14,] "GACUAGCUG"          "CUGAGCGAC"                       
## [15,] "CUCUGCUC"           "GAGUUGAG"                        
## [16,] "GCGUCCG"            "CGCUGGC"                         
## [17,] "GUCCGCG"            "CUGGCGC"                         
## [18,] "CGAUCAC"            "GCUUGUG"                         
## [19,] "GACUACCUG"          "CUGAGGGAC"                       
## [20,] "GACGAUCUG"          "CUGCCAGAC"                       
##       Environment.Nucleic acid concentration (M) Environment.Hybridization type
##  [1,] "4e-04"                                    "rnarna"                      
##  [2,] "4e-04"                                    "rnarna"                      
##  [3,] "4e-04"                                    "rnarna"                      
##  [4,] "4e-04"                                    "rnarna"                      
##  [5,] "4e-04"                                    "rnarna"                      
##  [6,] "4e-04"                                    "rnarna"                      
##  [7,] "4e-04"                                    "rnarna"                      
##  [8,] "4e-04"                                    "rnarna"                      
##  [9,] "4e-04"                                    "rnarna"                      
## [10,] "4e-04"                                    "rnarna"                      
## [11,] "4e-04"                                    "rnarna"                      
## [12,] "4e-04"                                    "rnarna"                      
## [13,] "4e-04"                                    "rnarna"                      
## [14,] "4e-04"                                    "rnarna"                      
## [15,] "4e-04"                                    "rnarna"                      
## [16,] "4e-04"                                    "rnarna"                      
## [17,] "4e-04"                                    "rnarna"                      
## [18,] "4e-04"                                    "rnarna"                      
## [19,] "4e-04"                                    "rnarna"                      
## [20,] "4e-04"                                    "rnarna"                      
##       Environment.Na concentration (M) Environment.Mg concentration (M)
##  [1,] "1"                              "0"                             
##  [2,] "1"                              "0"                             
##  [3,] "1"                              "0"                             
##  [4,] "1"                              "0"                             
##  [5,] "1"                              "0"                             
##  [6,] "1"                              "0"                             
##  [7,] "1"                              "0"                             
##  [8,] "1"                              "0"                             
##  [9,] "1"                              "0"                             
## [10,] "1"                              "0"                             
## [11,] "1"                              "0"                             
## [12,] "1"                              "0"                             
## [13,] "1"                              "0"                             
## [14,] "1"                              "0"                             
## [15,] "1"                              "0"                             
## [16,] "1"                              "0"                             
## [17,] "1"                              "0"                             
## [18,] "1"                              "0"                             
## [19,] "1"                              "0"                             
## [20,] "1"                              "0"                             
##       Environment.Tris concentration (M) Environment.K concentration (M)
##  [1,] "0"                                "0"                            
##  [2,] "0"                                "0"                            
##  [3,] "0"                                "0"                            
##  [4,] "0"                                "0"                            
##  [5,] "0"                                "0"                            
##  [6,] "0"                                "0"                            
##  [7,] "0"                                "0"                            
##  [8,] "0"                                "0"                            
##  [9,] "0"                                "0"                            
## [10,] "0"                                "0"                            
## [11,] "0"                                "0"                            
## [12,] "0"                                "0"                            
## [13,] "0"                                "0"                            
## [14,] "0"                                "0"                            
## [15,] "0"                                "0"                            
## [16,] "0"                                "0"                            
## [17,] "0"                                "0"                            
## [18,] "0"                                "0"                            
## [19,] "0"                                "0"                            
## [20,] "0"                                "0"                            
##       Environment.dNTP concentration (M) Environment.DMSO concentration (%)
##  [1,] "0"                                "0"                               
##  [2,] "0"                                "0"                               
##  [3,] "0"                                "0"                               
##  [4,] "0"                                "0"                               
##  [5,] "0"                                "0"                               
##  [6,] "0"                                "0"                               
##  [7,] "0"                                "0"                               
##  [8,] "0"                                "0"                               
##  [9,] "0"                                "0"                               
## [10,] "0"                                "0"                               
## [11,] "0"                                "0"                               
## [12,] "0"                                "0"                               
## [13,] "0"                                "0"                               
## [14,] "0"                                "0"                               
## [15,] "0"                                "0"                               
## [16,] "0"                                "0"                               
## [17,] "0"                                "0"                               
## [18,] "0"                                "0"                               
## [19,] "0"                                "0"                               
## [20,] "0"                                "0"                               
##       Environment.Formamide concentration (M or %)
##  [1,] "0"                                         
##  [2,] "0"                                         
##  [3,] "0"                                         
##  [4,] "0"                                         
##  [5,] "0"                                         
##  [6,] "0"                                         
##  [7,] "0"                                         
##  [8,] "0"                                         
##  [9,] "0"                                         
## [10,] "0"                                         
## [11,] "0"                                         
## [12,] "0"                                         
## [13,] "0"                                         
## [14,] "0"                                         
## [15,] "0"                                         
## [16,] "0"                                         
## [17,] "0"                                         
## [18,] "0"                                         
## [19,] "0"                                         
## [20,] "0"                                         
##       Environment.Self complementarity Environment.Correction factor
##  [1,] "FALSE"                          "4"                          
##  [2,] "FALSE"                          "4"                          
##  [3,] "FALSE"                          "4"                          
##  [4,] "FALSE"                          "4"                          
##  [5,] "FALSE"                          "4"                          
##  [6,] "FALSE"                          "4"                          
##  [7,] "FALSE"                          "4"                          
##  [8,] "FALSE"                          "4"                          
##  [9,] "FALSE"                          "4"                          
## [10,] "FALSE"                          "4"                          
## [11,] "FALSE"                          "4"                          
## [12,] "FALSE"                          "4"                          
## [13,] "FALSE"                          "4"                          
## [14,] "FALSE"                          "4"                          
## [15,] "FALSE"                          "4"                          
## [16,] "FALSE"                          "4"                          
## [17,] "FALSE"                          "4"                          
## [18,] "FALSE"                          "4"                          
## [19,] "FALSE"                          "4"                          
## [20,] "FALSE"                          "4"                          
##       Options.Approximative formula Options.Nearest neighbour model
##  [1,] NA                            "xia98"                        
##  [2,] NA                            "xia98"                        
##  [3,] NA                            "xia98"                        
##  [4,] NA                            "xia98"                        
##  [5,] NA                            "xia98"                        
##  [6,] NA                            "xia98"                        
##  [7,] NA                            "xia98"                        
##  [8,] NA                            "xia98"                        
##  [9,] NA                            "xia98"                        
## [10,] NA                            "xia98"                        
## [11,] NA                            "xia98"                        
## [12,] NA                            "xia98"                        
## [13,] NA                            "xia98"                        
## [14,] NA                            "xia98"                        
## [15,] NA                            "xia98"                        
## [16,] NA                            "xia98"                        
## [17,] NA                            "xia98"                        
## [18,] NA                            "xia98"                        
## [19,] NA                            "xia98"                        
## [20,] NA                            "xia98"                        
##       Options.GU model Options.Single mismatch model
##  [1,] "ser12"          "tur06"                      
##  [2,] "ser12"          "tur06"                      
##  [3,] "ser12"          "tur06"                      
##  [4,] "ser12"          "tur06"                      
##  [5,] "ser12"          "tur06"                      
##  [6,] "ser12"          "tur06"                      
##  [7,] "ser12"          "tur06"                      
##  [8,] "ser12"          "tur06"                      
##  [9,] "ser12"          "tur06"                      
## [10,] "ser12"          "tur06"                      
## [11,] "ser12"          "tur06"                      
## [12,] "ser12"          "tur06"                      
## [13,] "ser12"          "tur06"                      
## [14,] "ser12"          "tur06"                      
## [15,] "ser12"          "tur06"                      
## [16,] "ser12"          "tur06"                      
## [17,] "ser12"          "tur06"                      
## [18,] "ser12"          "tur06"                      
## [19,] "ser12"          "tur06"                      
## [20,] "ser12"          "tur06"                      
##       Options.Tandem mismatch model Options.Single dangling end model
##  [1,] "tur06"                       "ser08"                          
##  [2,] "tur06"                       "ser08"                          
##  [3,] "tur06"                       "ser08"                          
##  [4,] "tur06"                       "ser08"                          
##  [5,] "tur06"                       "ser08"                          
##  [6,] "tur06"                       "ser08"                          
##  [7,] "tur06"                       "ser08"                          
##  [8,] "tur06"                       "ser08"                          
##  [9,] "tur06"                       "ser08"                          
## [10,] "tur06"                       "ser08"                          
## [11,] "tur06"                       "ser08"                          
## [12,] "tur06"                       "ser08"                          
## [13,] "tur06"                       "ser08"                          
## [14,] "tur06"                       "ser08"                          
## [15,] "tur06"                       "ser08"                          
## [16,] "tur06"                       "ser08"                          
## [17,] "tur06"                       "ser08"                          
## [18,] "tur06"                       "ser08"                          
## [19,] "tur06"                       "ser08"                          
## [20,] "tur06"                       "ser08"                          
##       Options.Double dangling end model Options.Long dangling end model
##  [1,] "ser06"                           "sugrna02"                     
##  [2,] "ser06"                           "sugrna02"                     
##  [3,] "ser06"                           "sugrna02"                     
##  [4,] "ser06"                           "sugrna02"                     
##  [5,] "ser06"                           "sugrna02"                     
##  [6,] "ser06"                           "sugrna02"                     
##  [7,] "ser06"                           "sugrna02"                     
##  [8,] "ser06"                           "sugrna02"                     
##  [9,] "ser06"                           "sugrna02"                     
## [10,] "ser06"                           "sugrna02"                     
## [11,] "ser06"                           "sugrna02"                     
## [12,] "ser06"                           "sugrna02"                     
## [13,] "ser06"                           "sugrna02"                     
## [14,] "ser06"                           "sugrna02"                     
## [15,] "ser06"                           "sugrna02"                     
## [16,] "ser06"                           "sugrna02"                     
## [17,] "ser06"                           "sugrna02"                     
## [18,] "ser06"                           "sugrna02"                     
## [19,] "ser06"                           "sugrna02"                     
## [20,] "ser06"                           "sugrna02"                     
##       Options.Internal loop model Options.Single bulge loop model
##  [1,] "tur06"                     "tur06"                        
##  [2,] "tur06"                     "tur06"                        
##  [3,] "tur06"                     "tur06"                        
##  [4,] "tur06"                     "tur06"                        
##  [5,] "tur06"                     "tur06"                        
##  [6,] "tur06"                     "tur06"                        
##  [7,] "tur06"                     "tur06"                        
##  [8,] "tur06"                     "tur06"                        
##  [9,] "tur06"                     "tur06"                        
## [10,] "tur06"                     "tur06"                        
## [11,] "tur06"                     "tur06"                        
## [12,] "tur06"                     "tur06"                        
## [13,] "tur06"                     "tur06"                        
## [14,] "tur06"                     "tur06"                        
## [15,] "tur06"                     "tur06"                        
## [16,] "tur06"                     "tur06"                        
## [17,] "tur06"                     "tur06"                        
## [18,] "tur06"                     "tur06"                        
## [19,] "tur06"                     "tur06"                        
## [20,] "tur06"                     "tur06"                        
##       Options.Long bulge loop model Options.CNG repeats model
##  [1,] "tur06"                       NA                       
##  [2,] "tur06"                       NA                       
##  [3,] "tur06"                       NA                       
##  [4,] "tur06"                       NA                       
##  [5,] "tur06"                       NA                       
##  [6,] "tur06"                       NA                       
##  [7,] "tur06"                       NA                       
##  [8,] "tur06"                       NA                       
##  [9,] "tur06"                       NA                       
## [10,] "tur06"                       NA                       
## [11,] "tur06"                       NA                       
## [12,] "tur06"                       NA                       
## [13,] "tur06"                       NA                       
## [14,] "tur06"                       NA                       
## [15,] "tur06"                       NA                       
## [16,] "tur06"                       NA                       
## [17,] "tur06"                       NA                       
## [18,] "tur06"                       NA                       
## [19,] "tur06"                       NA                       
## [20,] "tur06"                       NA                       
##       Options.Inosine bases model Options.Hydroxyadenine bases model
##  [1,] "zno07"                     NA                                
##  [2,] "zno07"                     NA                                
##  [3,] "zno07"                     NA                                
##  [4,] "zno07"                     NA                                
##  [5,] "zno07"                     NA                                
##  [6,] "zno07"                     NA                                
##  [7,] "zno07"                     NA                                
##  [8,] "zno07"                     NA                                
##  [9,] "zno07"                     NA                                
## [10,] "zno07"                     NA                                
## [11,] "zno07"                     NA                                
## [12,] "zno07"                     NA                                
## [13,] "zno07"                     NA                                
## [14,] "zno07"                     NA                                
## [15,] "zno07"                     NA                                
## [16,] "zno07"                     NA                                
## [17,] "zno07"                     NA                                
## [18,] "zno07"                     NA                                
## [19,] "zno07"                     NA                                
## [20,] "zno07"                     NA                                
##       Options.Azobenzenes model Options.Locked nucleic acids model
##  [1,] NA                        NA                                
##  [2,] NA                        NA                                
##  [3,] NA                        NA                                
##  [4,] NA                        NA                                
##  [5,] NA                        NA                                
##  [6,] NA                        NA                                
##  [7,] NA                        NA                                
##  [8,] NA                        NA                                
##  [9,] NA                        NA                                
## [10,] NA                        NA                                
## [11,] NA                        NA                                
## [12,] NA                        NA                                
## [13,] NA                        NA                                
## [14,] NA                        NA                                
## [15,] NA                        NA                                
## [16,] NA                        NA                                
## [17,] NA                        NA                                
## [18,] NA                        NA                                
## [19,] NA                        NA                                
## [20,] NA                        NA                                
##       Options.Ion correction method Options.Na equivalence correction method
##  [1,] NA                            "ahs01"                                 
##  [2,] NA                            "ahs01"                                 
##  [3,] NA                            "ahs01"                                 
##  [4,] NA                            "ahs01"                                 
##  [5,] NA                            "ahs01"                                 
##  [6,] NA                            "ahs01"                                 
##  [7,] NA                            "ahs01"                                 
##  [8,] NA                            "ahs01"                                 
##  [9,] NA                            "ahs01"                                 
## [10,] NA                            "ahs01"                                 
## [11,] NA                            "ahs01"                                 
## [12,] NA                            "ahs01"                                 
## [13,] NA                            "ahs01"                                 
## [14,] NA                            "ahs01"                                 
## [15,] NA                            "ahs01"                                 
## [16,] NA                            "ahs01"                                 
## [17,] NA                            "ahs01"                                 
## [18,] NA                            "ahs01"                                 
## [19,] NA                            "ahs01"                                 
## [20,] NA                            "ahs01"                                 
##       Options.DMSO correction method Options.Formamide correction method
##  [1,] "ahs01"                        "bla96"                            
##  [2,] "ahs01"                        "bla96"                            
##  [3,] "ahs01"                        "bla96"                            
##  [4,] "ahs01"                        "bla96"                            
##  [5,] "ahs01"                        "bla96"                            
##  [6,] "ahs01"                        "bla96"                            
##  [7,] "ahs01"                        "bla96"                            
##  [8,] "ahs01"                        "bla96"                            
##  [9,] "ahs01"                        "bla96"                            
## [10,] "ahs01"                        "bla96"                            
## [11,] "ahs01"                        "bla96"                            
## [12,] "ahs01"                        "bla96"                            
## [13,] "ahs01"                        "bla96"                            
## [14,] "ahs01"                        "bla96"                            
## [15,] "ahs01"                        "bla96"                            
## [16,] "ahs01"                        "bla96"                            
## [17,] "ahs01"                        "bla96"                            
## [18,] "ahs01"                        "bla96"                            
## [19,] "ahs01"                        "bla96"                            
## [20,] "ahs01"                        "bla96"                            
##       Options.Mode Results.Enthalpy (cal) Results.Entropy (cal)
##  [1,] "def"        "-47650"               "-138.8"             
##  [2,] "def"        "-71130"               "-200.5"             
##  [3,] "def"        "-57930"               "-164.1"             
##  [4,] "def"        "-60570"               "-176.6"             
##  [5,] "def"        "-79870"               "-219.9"             
##  [6,] "def"        "-68380"               "-194.5"             
##  [7,] "def"        "-73880"               "-208.3"             
##  [8,] "def"        "-78430"               "-218.3"             
##  [9,] "def"        "-59650"               "-171.5"             
## [10,] "def"        "-61330"               "-173.8"             
## [11,] "def"        "-58350"               "-165.4"             
## [12,] "def"        "-64570"               "-184.7"             
## [13,] "def"        "-70970"               "-200.6"             
## [14,] "def"        "-72010"               "-203"               
## [15,] "def"        "-58820"               "-171"               
## [16,] "def"        "-59840"               "-169.6"             
## [17,] "def"        "-59840"               "-169.6"             
## [18,] "def"        "-50210"               "-148.3"             
## [19,] "def"        "-70520"               "-198.8"             
## [20,] "def"        "-69730"               "-198.2"             
##       Results.Enthalpy (J) Results.Entropy (J) Results.Melting temperature (C)
##  [1,] "-199177"            "-580.184"          "30.1048299398322"             
##  [2,] "-297323.4"          "-838.09"           "51.8989532242754"             
##  [3,] "-242147.4"          "-685.938"          "44.3989325444856"             
##  [4,] "-253182.6"          "-738.188"          "37.5791954133529"             
##  [5,] "-333856.6"          "-919.182"          "62.1162425798375"             
##  [6,] "-285828.4"          "-813.01"           "48.141439592185"              
##  [7,] "-308818.4"          "-870.694"          "52.845957204839"              
##  [8,] "-327837.4"          "-912.494"          "58.2977096620104"             
##  [9,] "-249337"            "-716.87"           "41.08087522322"               
## [10,] "-256359.4"          "-726.484"          "46.0633145887674"             
## [11,] "-243903"            "-691.372"          "44.4380466975271"             
## [12,] "-269902.6"          "-772.046"          "44.8840464672343"             
## [13,] "-296654.6"          "-838.508"          "51.0196486690585"             
## [14,] "-301001.8"          "-848.54"           "52.203376709706"              
## [15,] "-245867.6"          "-714.78"           "37.5268181873443"             
## [16,] "-250131.2"          "-708.928"          "45.2688421309843"             
## [17,] "-250131.2"          "-708.928"          "45.2688421309843"             
## [18,] "-209877.8"          "-619.894"          "28.1788644808993"             
## [19,] "-294773.6"          "-830.984"          "51.6345164549562"             
## [20,] "-291471.4"          "-828.476"          "48.8860141674642"             
##       Message
##  [1,] NA     
##  [2,] NA     
##  [3,] NA     
##  [4,] NA     
##  [5,] NA     
##  [6,] NA     
##  [7,] NA     
##  [8,] NA     
##  [9,] NA     
## [10,] NA     
## [11,] NA     
## [12,] NA     
## [13,] NA     
## [14,] NA     
## [15,] NA     
## [16,] NA     
## [17,] NA     
## [18,] NA     
## [19,] NA     
## [20,] NA</code></pre>
</div>
<div id="further-reading" class="section level1">
<h1 class="hasAnchor">
<a href="#further-reading" class="anchor"></a>Further reading</h1>
<p>Further details about algorithm, formulae and methods are available in the <a href="https://www.ebi.ac.uk/biomodels-static/tools/melting/melting5-doc/melting.html">MELTING 5 documentation</a>.</p>
</div>
<div id="citing-rmelting" class="section level1">
<h1 class="hasAnchor">
<a href="#citing-rmelting" class="anchor"></a>Citing <code>rmelting</code>
</h1>
<pre><code>## 
## To cite the R package 'rmelting' in publications use:
## 
##   Aravind, J. and Krishna, G. K. (2021).  rmelting: R Interface to
##   MELTING 5. R package version 1.7.1,
##   https://aravind-j.github.io/rmelting/.
## 
## A BibTeX entry for LaTeX users is
## 
##   @Manual{,
##     title = {rmelting: R Interface to MELTING 5},
##     author = {J. Aravind and G. K. Krishna},
##     year = {2021},
##     note = {R package version 1.7.1},
##     note = {https://aravind-j.github.io/rmelting/},
##   }
## 
## This free and open-source software implements academic research by the
## authors and co-workers. If you use it, please support the project by
## citing the package.</code></pre>
</div>
<div id="session-info" class="section level1">
<h1 class="hasAnchor">
<a href="#session-info" class="anchor"></a>Session Info</h1>
<div class="sourceCode" id="cb317"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/r/utils/sessionInfo.html">sessionInfo</a></span><span class="op">(</span><span class="op">)</span></code></pre></div>
<pre><code>## R version 4.0.4 (2021-02-15)
## Platform: x86_64-apple-darwin17.0 (64-bit)
## Running under: macOS Catalina 10.15.7
## 
## Matrix products: default
## BLAS:   /Library/Frameworks/R.framework/Versions/4.0/Resources/lib/libRblas.dylib
## LAPACK: /Library/Frameworks/R.framework/Versions/4.0/Resources/lib/libRlapack.dylib
## 
## locale:
## [1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8
## 
## attached base packages:
## [1] stats     graphics  grDevices utils     datasets  methods   base     
## 
## other attached packages:
## [1] rmelting_1.7.1 readxl_1.3.1  
## 
## loaded via a namespace (and not attached):
##  [1] Rcpp_1.0.6        knitr_1.31        magrittr_2.0.1    R6_2.5.0         
##  [5] ragg_1.1.1        rlang_0.4.10      fastmap_1.1.0     fansi_0.4.2      
##  [9] highr_0.8         stringr_1.4.0     rbibutils_2.0     tools_4.0.4      
## [13] xfun_0.21         utf8_1.1.4        ellipsis_0.3.1    htmltools_0.5.1.1
## [17] systemfonts_1.0.1 yaml_2.2.1        assertthat_0.2.1  digest_0.6.27    
## [21] rprojroot_2.0.2   lifecycle_1.0.0   tibble_3.1.0      pkgdown_1.6.1    
## [25] crayon_1.4.1      textshaping_0.3.1 rJava_0.9-13      vctrs_0.3.6      
## [29] Rdpack_2.1.1      fs_1.5.0          memoise_2.0.0     cachem_1.0.4     
## [33] evaluate_0.14     rmarkdown_2.7     stringi_1.5.3     pander_0.6.3     
## [37] pillar_1.5.0      compiler_4.0.4    cellranger_1.1.0  desc_1.2.0       
## [41] pkgconfig_2.0.3</code></pre>
</div>
<div id="references" class="section level1 unnumbered">
<h1 class="hasAnchor">
<a href="#references" class="anchor"></a>References</h1>
<div id="refs" class="references">
<div id="ref-von_ahsen_oligonucleotide_2001">
<p>Ahsen, N. von, Wittwer, C. T., and Schütz, E. (2001). Oligonucleotide melting temperatures under PCR conditions: Nearest-neighbor corrections for Mg2+, deoxynucleotide triphosphate, and dimethyl sulfoxide concentrations with comparison to alternative empirical formulas. <em>Clinical Chemistry</em> 47, 1956–1961. Available at: <a href="http://clinchem.aaccjnls.org/content/47/11/1956">http://clinchem.aaccjnls.org/content/47/11/1956</a>.</p>
</div>
<div id="ref-allawi_thermodynamics_1997">
<p>Allawi, H. T., and SantaLucia, J. (1997). Thermodynamics and NMR of internal G·T mismatches in dna. <em>Biochemistry</em> 36, 10581–10594. doi:<a href="https://doi.org/10.1021/bi962590c">10.1021/bi962590c</a>.</p>
</div>
<div id="ref-allawi_nearest_1998">
<p>Allawi, H. T., and SantaLucia, J. (1998a). Nearest neighbor thermodynamic parameters for internal G·A mismatches in DNA. <em>Biochemistry</em> 37, 2170–2179. doi:<a href="https://doi.org/10.1021/bi9724873">10.1021/bi9724873</a>.</p>
</div>
<div id="ref-allawi_nearest-neighbor_1998">
<p>Allawi, H. T., and SantaLucia, J. (1998b). Nearest-neighbor thermodynamics of internal A·C mismatches in dna: Sequence dependence and pH effects. <em>Biochemistry</em> 37, 9435–9444. doi:<a href="https://doi.org/10.1021/bi9803729">10.1021/bi9803729</a>.</p>
</div>
<div id="ref-allawi_thermodynamics_1998">
<p>Allawi, H. T., and SantaLucia, J. (1998c). Thermodynamics of internal C·T mismatches in DNA. <em>Nucleic Acids Research</em> 26, 2694–2701. doi:<a href="https://doi.org/10.1093/nar/26.11.2694">10.1093/nar/26.11.2694</a>.</p>
</div>
<div id="ref-asanuma_clear-cut_2005">
<p>Asanuma, H., Matsunaga, D., and Komiyama, M. (2005). Clear-cut photo-regulation of the formation and dissociation of the DNA duplex by modified oligonucleotide involving multiple azobenzenes. <em>Nucleic Acids Symposium Series</em>, 35–36. doi:<a href="https://doi.org/10.1093/nass/49.1.35">10.1093/nass/49.1.35</a>.</p>
</div>
<div id="ref-badhwar_thermodynamic_2007">
<p>Badhwar, J., Karri, S., Cass, C. K., Wunderlich, E. L., and Znosko, B. M. (2007). Thermodynamic characterization of RNA duplexes containing naturally occurring 1 × 2 nucleotide internal loops. <em>Biochemistry</em> 46, 14715–14724. doi:<a href="https://doi.org/10.1021/bi701024w">10.1021/bi701024w</a>.</p>
</div>
<div id="ref-blake_denaturation_1996">
<p>Blake, R. D. (1996). “Denaturation of DNA,” in <em>Encyclopedia of molecular biology and molecular medicine</em>, ed. R. A. Meyers (Weinheim, Germany: VCH Verlagsgesellschaft), 1–19.</p>
</div>
<div id="ref-blake_thermal_1998">
<p>Blake, R. D., and Delcourt, S. G. (1998). Thermal stability of DNA. <em>Nucleic Acids Research</em> 26, 3323–3332. doi:<a href="https://doi.org/10.1093/nar/26.14.3323">10.1093/nar/26.14.3323</a>.</p>
</div>
<div id="ref-blose_non-nearest-neighbor_2007">
<p>Blose, J. M., Manni, M. L., Klapec, K. A., Stranger-Jones, Y., Zyra, A. C., Sim, V., et al. (2007). Non-nearest-neighbor dependence of stability for RNA bulge loops based on the complete set of group i single nucleotide bulge loops. <em>Biochemistry</em> 46, 15123–15135. doi:<a href="https://doi.org/10.1021/bi700736f">10.1021/bi700736f</a>.</p>
</div>
<div id="ref-bommarito_thermodynamic_2000">
<p>Bommarito, S., Peyret, N., and SantaLucia, J. (2000). Thermodynamic parameters for DNA sequences with dangling ends. <em>Nucleic Acids Research</em> 28, 1929–1934. doi:<a href="https://doi.org/10.1093/nar/28.9.1929">10.1093/nar/28.9.1929</a>.</p>
</div>
<div id="ref-breslauer_predicting_1986">
<p>Breslauer, K. J., Frank, R., Blöcker, H., and Marky, L. A. (1986). Predicting DNA duplex stability from the base sequence. <em>Proceedings of the National Academy of Sciences</em> 83, 3746. doi:<a href="https://doi.org/10.1073/pnas.83.11.3746">10.1073/pnas.83.11.3746</a>.</p>
</div>
<div id="ref-britten_analysis_1974">
<p>Britten, R. J., Graham, D. E., and Neufeld, B. R. (1974). Analysis of repeating DNA sequences by reassociation. <em>Methods in Enzymology</em> 29, 363–418. doi:<a href="https://doi.org/10.1016/0076-6879(74)29033-5">10.1016/0076-6879(74)29033-5</a>.</p>
</div>
<div id="ref-broda_thermodynamic_2005">
<p>Broda, M., Kierzek, E., Gdaniec, Z., Kulinski, T., and Kierzek, R. (2005). Thermodynamic stability of RNA structures formed by CNG trinucleotide repeats. Implication for prediction of RNA structure. <em>Biochemistry</em> 44, 10873–10882. doi:<a href="https://doi.org/10.1021/bi0502339">10.1021/bi0502339</a>.</p>
</div>
<div id="ref-casey_rates_1977">
<p>Casey, J., and Davidson, N. (1977). Rates of formation and thermal stabilities of RNA:DNA and DNA:DNA duplexes at high concentrations of formamide. <em>Nucleic Acids Research</em> 4, 1539–1552. doi:<a href="https://doi.org/10.1093/nar/4.5.1539">10.1093/nar/4.5.1539</a>.</p>
</div>
<div id="ref-chen_testing_2012">
<p>Chen, J. L., Dishler, A. L., Kennedy, S. D., Yildirim, I., Liu, B., Turner, D. H., et al. (2012). Testing the nearest neighbor model for canonical rna base pairs: Revision of GU parameters. <em>Biochemistry</em> 51, 3508–3522. doi:<a href="https://doi.org/10.1021/bi3002709">10.1021/bi3002709</a>.</p>
</div>
<div id="ref-chester_dimethyl_1993">
<p>Chester, N., and Marshak, D. R. (1993). Dimethyl sulfoxide-mediated primer Tm reduction: A method for analyzing the role of renaturation temperature in the polymerase chain reaction. <em>Analytical Biochemistry</em> 209, 284–290. doi:<a href="https://doi.org/10.1006/abio.1993.1121">10.1006/abio.1993.1121</a>.</p>
</div>
<div id="ref-cullen_thermal_1976">
<p>Cullen, B. R., and Bick, M. D. (1976). Thermal denaturation of DNA from bromodeoxyuridine substituted cells. <em>Nucleic Acids Research</em> 3, 49–62. doi:<a href="https://doi.org/10.1093/nar/3.1.49">10.1093/nar/3.1.49</a>.</p>
</div>
<div id="ref-davis_thermodynamic_2007">
<p>Davis, A. R., and Znosko, B. M. (2007). Thermodynamic characterization of single mismatches found in naturally occurring RNA. <em>Biochemistry</em> 46, 13425–13436. doi:<a href="https://doi.org/10.1021/bi701311c">10.1021/bi701311c</a>.</p>
</div>
<div id="ref-davis_thermodynamic_2008">
<p>Davis, A. R., and Znosko, B. M. (2008). Thermodynamic characterization of naturally occurring RNA single mismatches with G-U nearest neighbors. <em>Biochemistry</em> 47, 10178–10187. doi:<a href="https://doi.org/10.1021/bi800471z">10.1021/bi800471z</a>.</p>
</div>
<div id="ref-dumousseau_melting_2012">
<p>Dumousseau, M., Rodriguez, N., Juty, N., and Le Novère, N. (2012). MELTING, a flexible platform to predict the melting temperatures of nucleic acids. <em>BMC Bioinformatics</em> 13, 101. doi:<a href="https://doi.org/10.1186/1471-2105-13-101">10.1186/1471-2105-13-101</a>.</p>
</div>
<div id="ref-escara_thermal_1980">
<p>Escara, J. F., and Hutton, J. R. (1980). Thermal stability and renaturation of DNA in dimethyl sulfoxide solutions: Acceleration of the renaturation rate. <em>Biopolymers</em> 19, 1315–1327. doi:<a href="https://doi.org/10.1002/bip.1980.360190708">10.1002/bip.1980.360190708</a>.</p>
</div>
<div id="ref-frank-kamenetskii_simplification_1971">
<p>Frank-Kamenetskii, M. D. (1971). Simplification of the empirical relationship between melting temperature of DNA, its GC content and concentration of sodium ions in solution. <em>Biopolymers</em> 10, 2623–2624. doi:<a href="https://doi.org/10.1002/bip.360101223">10.1002/bip.360101223</a>.</p>
</div>
<div id="ref-freier_improved_1986">
<p>Freier, S. M., Kierzek, R., Jaeger, J. A., Sugimoto, N., Caruthers, M. H., Neilson, T., et al. (1986). Improved free-energy parameters for predictions of RNA duplex stability. <em>Proceedings of the National Academy of Sciences</em> 83, 9373. doi:<a href="https://doi.org/10.1073/pnas.83.24.9373">10.1073/pnas.83.24.9373</a>.</p>
</div>
<div id="ref-hall_evolution_1980">
<p>Hall, T. J., Grula, J. W., Davidson, E. H., and Britten, R. J. (1980). Evolution of sea urchin non-repetitive DNA. <em>Journal of Molecular Evolution</em> 16, 95–110. doi:<a href="https://doi.org/10.1007/BF01731580">10.1007/BF01731580</a>.</p>
</div>
<div id="ref-hutton_renaturation_1977">
<p>Hutton, J. R. (1977). Renaturation kinetics and thermal stability of DNA in aqueous solutions of formamide and urea. <em>Nucleic Acids Research</em> 4, 3537–3555. doi:<a href="https://doi.org/10.1093/nar/4.10.3537">10.1093/nar/4.10.3537</a>.</p>
</div>
<div id="ref-kawakami_thermodynamic_2001">
<p>Kawakami, J., Kamiya, H., Yasuda, K., Fujiki, H., Kasai, H., and Sugimoto, N. (2001). Thermodynamic stability of base pairs between 2-hydroxyadenine and incoming nucleotides as a determinant of nucleotide incorporation specificity during replication. <em>Nucleic Acids Research</em> 29, 3289–3296. doi:<a href="https://doi.org/10.1093/nar/29.16.3289">10.1093/nar/29.16.3289</a>.</p>
</div>
<div id="ref-kierzek_nearest_2006">
<p>Kierzek, E., Mathews, D. H., Ciesielska, A., Turner, D. H., and Kierzek, R. (2006). Nearest neighbor parameters for Watson-Crick complementary heteroduplexes formed between 2’-O-methyl RNA and RNA oligonucleotides. <em>Nucleic Acids Research</em> 34, 3609–3614. doi:<a href="https://doi.org/10.1093/nar/gkl232">10.1093/nar/gkl232</a>.</p>
</div>
<div id="ref-le_novere_melting_2001">
<p>Le Novère, N. (2001). MELTING, computing the melting temperature of nucleic acid duplex. <em>Bioinformatics</em> 17, 1226–1227. doi:<a href="https://doi.org/10.1093/bioinformatics/17.12.1226">10.1093/bioinformatics/17.12.1226</a>.</p>
</div>
<div id="ref-lu_set_2006">
<p>Lu, Z. J., Turner, D. H., and Mathews, D. H. (2006). A set of nearest neighbor parameters for predicting the enthalpy change of RNA secondary structure formation. <em>Nucleic Acids Research</em> 34, 4912–4924. doi:<a href="https://doi.org/10.1093/nar/gkl472">10.1093/nar/gkl472</a>.</p>
</div>
<div id="ref-marmur_determination_1962">
<p>Marmur, J., and Doty, P. (1962). Determination of the base composition of deoxyribonucleic acid from its thermal denaturation temperature. <em>Journal of Molecular Biology</em> 5, 109–118. doi:<a href="https://doi.org/10.1016/S0022-2836(62)80066-7">10.1016/S0022-2836(62)80066-7</a>.</p>
</div>
<div id="ref-mathews_expanded_1999">
<p>Mathews, D. H., Sabina, J., Zuker, M., and Turner, D. H. (1999). Expanded sequence dependence of thermodynamic parameters improves prediction of RNA secondary structure. <em>Journal of Molecular Biology</em> 288, 911–940. doi:<a href="https://doi.org/10.1006/jmbi.1999.2700">10.1006/jmbi.1999.2700</a>.</p>
</div>
<div id="ref-mcconaughy_nucleic_1969">
<p>McConaughy, B. L., Laird, C., and McCarthy, B. J. (1969). Nucleic acid reassociation in formamide. <em>Biochemistry</em> 8, 3289–3295. doi:<a href="https://doi.org/10.1021/bi00836a024">10.1021/bi00836a024</a>.</p>
</div>
<div id="ref-mctigue_sequence-dependent_2004">
<p>McTigue, P. M., Peterson, R. J., and Kahn, J. D. (2004). Sequence-dependent thermodynamic parameters for locked nucleic acid (LNA)-DNA duplex formation. <em>Biochemistry</em> 43, 5388–5405. doi:<a href="https://doi.org/10.1021/bi035976d">10.1021/bi035976d</a>.</p>
</div>
<div id="ref-miller_thermodynamic_2008">
<p>Miller, S., Jones, L. E., Giovannitti, K., Piper, D., and Serra, M. J. (2008). Thermodynamic analysis of 5’ and 3’ single- and 3’ double-nucleotide overhangs neighboring wobble terminal base pairs. <em>Nucleic Acids Research</em> 36, 5652–5659. doi:<a href="https://doi.org/10.1093/nar/gkn525">10.1093/nar/gkn525</a>.</p>
</div>
<div id="ref-mitsuhashi_technical_1996">
<p>Mitsuhashi, M. (1996). Technical report: Part 1. Basic requirements for designing optimal oligonucleotide probe sequences. <em>Journal of Clinical Laboratory Analysis</em> 10, 277–284. doi:<a href="https://doi.org/10/cw9bn6">10/cw9bn6</a>.</p>
</div>
<div id="ref-musielski_influence_1981">
<p>Musielski, H., Mann, W., Laue, R., and Michel, S. (1981). Influence of dimethylsulfoxide on transcription by bacteriophage T3-induced RNA polymerase. <em>Zeitschrift für allgemeine Mikrobiologie</em> 21, 447–456. doi:<a href="https://doi.org/10.1002/jobm.19810210606">10.1002/jobm.19810210606</a>.</p>
</div>
<div id="ref-ohmichi_long_2002">
<p>Ohmichi, T., Nakano, S.-i., Miyoshi, D., and Sugimoto, N. (2002). Long RNA dangling end has large energetic contribution to duplex stability. <em>Journal of the American Chemical Society</em> 124, 10367–10372. doi:<a href="https://doi.org/10.1021/ja0255406">10.1021/ja0255406</a>.</p>
</div>
<div id="ref-otoole_comprehensive_2006">
<p>O’Toole, A. S., Miller, S., Haines, N., Zink, M. C., and Serra, M. J. (2006). Comprehensive thermodynamic analysis of 3’ double-nucleotide overhangs neighboring Watson-Crick terminal base pairs. <em>Nucleic Acids Research</em> 34, 3338–3344. doi:<a href="https://doi.org/10.1093/nar/gkl428">10.1093/nar/gkl428</a>.</p>
</div>
<div id="ref-otoole_stability_2005">
<p>O’Toole, A. S., Miller, S., and Serra, M. J. (2005). Stability of 3’ double nucleotide overhangs that model the 3’ ends of siRNA. <em>RNA</em> 11, 512–516. doi:<a href="https://doi.org/10.1261/rna.7254905">10.1261/rna.7254905</a>.</p>
</div>
<div id="ref-owczarzy_predicting_2008">
<p>Owczarzy, R., Moreira, B. G., You, Y., Behlke, M. A., and Walder, J. A. (2008). Predicting stability of DNA duplexes in solutions containing magnesium and monovalent cations. <em>Biochemistry</em> 47, 5336–5353. doi:<a href="https://doi.org/10.1021/bi702363u">10.1021/bi702363u</a>.</p>
</div>
<div id="ref-owczarzy_effects_2004">
<p>Owczarzy, R., You, Y., Moreira, B. G., Manthey, J. A., Huang, L., Behlke, M. A., et al. (2004). Effects of sodium ions on DNA duplex oligomers: Improved predictions of melting temperatures. <em>Biochemistry</em> 43, 3537–3554. doi:<a href="https://doi.org/10.1021/bi034621r">10.1021/bi034621r</a>.</p>
</div>
<div id="ref-owen_determination_1969">
<p>Owen, R., Hill, L., and Lapage, S. (1969). Determination of DNA base compositions from melting profiles in dilute buffers. <em>Biopolymers</em> 7, 503–516. doi:<a href="https://doi.org/10.1002/bip.1969.360070408">10.1002/bip.1969.360070408</a>.</p>
</div>
<div id="ref-peyret_prediction_2000">
<p>Peyret, N. (2000). Prediction of nucleic acid hybridization: Parameters and algorithms. Available at: <a href="http://elibrary.wayne.edu/record=2760965">http://elibrary.wayne.edu/record=2760965</a>.</p>
</div>
<div id="ref-peyret_nearest-neighbor_1999">
<p>Peyret, N., Seneviratne, P. A., Allawi, H. T., and SantaLucia, J. (1999). Nearest-Neighbor Thermodynamics and NMR of DNA Sequences with Internal A·A, C·C, G·G, and T·T Mismatches. <em>Biochemistry</em> 38, 3468–3477. doi:<a href="https://doi.org/10.1021/bi9825091">10.1021/bi9825091</a>.</p>
</div>
<div id="ref-record_electrostatic_1967">
<p>Record, M. T. (1967). Electrostatic effects on polynucleotide transitions. I. Behavior at neutral pH. <em>Biopolymers</em> 5, 975–992. doi:<a href="https://doi.org/10.1002/bip.1967.360051010">10.1002/bip.1967.360051010</a>.</p>
</div>
<div id="ref-santalucia_unified_1998">
<p>SantaLucia, J. (1998). A unified view of polymer, dumbbell, and oligonucleotide DNA nearest-neighbor thermodynamics. <em>Proceedings of the National Academy of Sciences</em> 95, 1460. doi:<a href="https://doi.org/10.1073/pnas.95.4.1460">10.1073/pnas.95.4.1460</a>.</p>
</div>
<div id="ref-santalucia_thermodynamics_2004">
<p>SantaLucia, J., and Hicks, D. (2004). The thermodynamics of DNA structural motifs. <em>Annual Review of Biophysics and Biomolecular Structure</em> 33, 415–440. doi:<a href="https://doi.org/10.1146/annurev.biophys.32.110601.141800">10.1146/annurev.biophys.32.110601.141800</a>.</p>
</div>
<div id="ref-santalucia_improved_1996">
<p>SantaLucia, John, Allawi, H. T., and Seneviratne, P. A. (1996). Improved nearest-neighbor parameters for predicting DNA duplex stability. <em>Biochemistry</em> 35, 3555–3562. doi:<a href="https://doi.org/10.1021/bi951907q">10.1021/bi951907q</a>.</p>
</div>
<div id="ref-schildkraut_dependence_1965">
<p>Schildkraut, C., and Lifson, S. (1965). Dependence of the melting temperature of DNA on salt concentration. <em>Biopolymers</em> 3, 195–208. doi:<a href="https://doi.org/10.1002/bip.360030207">10.1002/bip.360030207</a>.</p>
</div>
<div id="ref-sugimoto_improved_1996">
<p>Sugimoto, N., Nakano, S., Yoneyama, M., and Honda, K. (1996). Improved thermodynamic parameters and helix initiation factor to predict stability of DNA duplexes. <em>Nucleic Acids Research</em> 24, 4501–4505. doi:<a href="https://doi.org/10.1093/nar/24.22.4501">10.1093/nar/24.22.4501</a>.</p>
</div>
<div id="ref-tan_nucleic_2006">
<p>Tan, Z.-J., and Chen, S.-J. (2006). Nucleic acid helix stability: Effects of salt concentration, cation valence and size, and chain length. <em>Biophysical Journal</em> 90, 1175–1190. doi:<a href="https://doi.org/10.1529/biophysj.105.070904">10.1529/biophysj.105.070904</a>.</p>
</div>
<div id="ref-tan_rna_2007">
<p>Tan, Z.-J., and Chen, S.-J. (2007). RNA helix stability in mixed Na(+)/Mg(2+) solution. <em>Biophysical Journal</em> 92, 3615–3632. doi:<a href="https://doi.org/10.1529/biophysj.106.100388">10.1529/biophysj.106.100388</a>.</p>
</div>
<div id="ref-tanaka_thermodynamic_2004">
<p>Tanaka, F., Kameda, A., Yamamoto, M., and Ohuchi, A. (2004). Thermodynamic parameters based on a nearest-neighbor model for DNA sequences with a single-bulge loop. <em>Biochemistry</em> 43, 7143–7150. doi:<a href="https://doi.org/10.1021/bi036188r">10.1021/bi036188r</a>.</p>
</div>
<div id="ref-wahl_molecular_1987">
<p>Wahl, G. M., Barger, S. L., and Kimmel, A. R. (1987). Molecular hybridization of immobilized nucleic acids: Theoretical concepts and practical considerations. <em>Methods in Enzymology</em> 152, 399–407. doi:<a href="https://doi.org/10.1016/0076-6879(87)52046-8">10.1016/0076-6879(87)52046-8</a>.</p>
</div>
<div id="ref-watkins_thermodynamic_2011">
<p>Watkins, N. E., Kennelly, W. J., Tsay, M. J., Tuin, A., Swenson, L., Lee, H.-R., et al. (2011). Thermodynamic contributions of single internal rA·dA, rC·dC, rG·dG and rU·dT mismatches in RNA/DNA duplexes. <em>Nucleic Acids Research</em> 39, 1894–1902. doi:<a href="https://doi.org/10/cdm4jh">10/cdm4jh</a>.</p>
</div>
<div id="ref-watkins_nearest-neighbor_2005">
<p>Watkins, N. E., and SantaLucia, J. (2005). Nearest-neighbor thermodynamics of deoxyinosine pairs in DNA duplexes. <em>Nucleic Acids Research</em> 33, 6258–6267. doi:<a href="https://doi.org/10.1093/nar/gki918">10.1093/nar/gki918</a>.</p>
</div>
<div id="ref-wetmur_dna_1991">
<p>Wetmur, J. G. (1991). DNA probes: Applications of the principles of nucleic acid hybridization. <em>Critical Reviews in Biochemistry and Molecular Biology</em> 26, 227–259. doi:<a href="https://doi.org/10.3109/10409239109114069">10.3109/10409239109114069</a>.</p>
</div>
<div id="ref-wright_nearest_2007">
<p>Wright, D. J., Rice, J. L., Yanker, D. M., and Znosko, B. M. (2007). Nearest neighbor parameters for inosine·uridine pairs in RNA duplexes. <em>Biochemistry</em> 46, 4625–4634. doi:<a href="https://doi.org/10.1021/bi0616910">10.1021/bi0616910</a>.</p>
</div>
<div id="ref-xia_thermodynamic_1998">
<p>Xia, T., SantaLucia, J., Burkard, M. E., Kierzek, R., Schroeder, S. J., Jiao, X., et al. (1998). Thermodynamic parameters for an expanded nearest-neighbor model for formation of RNA duplexes with Watson-Crick base pairs. <em>Biochemistry</em> 37, 14719–14735. doi:<a href="https://doi.org/10.1021/bi9809425">10.1021/bi9809425</a>.</p>
</div>
</div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

        <nav id="toc" data-toggle="toc"><h2 data-toc-skip>Contents</h2>
    </nav>
</div>

</div>



      <footer><div class="copyright">
  <p>Developed by J. Aravind, G. K. Krishna.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.6.1.</p>
</div>

      </footer>
</div>

  
<script src="https://cdnjs.cloudflare.com/ajax/libs/docsearch.js/2.6.1/docsearch.min.js" integrity="sha256-GKvGqXDznoRYHCwKXGnuchvKSwmx9SRMrZOTh2g4Sb0=" crossorigin="anonymous"></script><script>
  docsearch({
    
    
    apiKey: '8b84820d42f7497ca93d51aa44ae523f',
    indexName: 'rmelting',
    inputSelector: 'input#search-input.form-control',
    transformData: function(hits) {
      return hits.map(function (hit) {
        hit.url = updateHitURL(hit);
        return hit;
      });
    }
  });
</script>
</body>
</html>
